## ADMET
- **Computation-ADME**: 
	- [Code](https://github.com/molecularinformatics/Computational-ADME) : Last updated in February 2024, 3 months ago

## Chemical
- **ChemBL**: A manually curated database of bioactive molecules with drug-like properties, integrating chemical, bioactivity, and genomic data to aid drug discovery.
	- [Publication](https://doi.org/10.1093/nar/gkr777) : Citations: 2818.0
	- [Link](https://www.ebi.ac.uk/chembl/)
- **ChemMine**: A compound mining portal to facilitate drug and agrochemical discovery, offering chemical genomics screens and a toolbox for cheminformatics research.
	- [Publication](https://doi.org/10.1104/pp.105.062687) : Citations: 50.0
	- [Link](http://chemminedb.ucr.edu/)
- **ChemSpider**: commercially-available compounds
	- [Publication](https://doi.org/10.1021/ed100697w) : Citations: 813.0
	- [Link](http://www.chemspider.com/)
- **DrugBank**: Offers structured drug information, including emerging research and novel connections, to support drug discovery, clinical software, and academic research.
	- [Publication](https://doi.org/10.1093%2Fnar%2Fgkm958) : Citations: 2050.0
	- [Link](https://go.drugbank.com/)
- **DrugMAP**: a comprehensive structural resource of protein complexes (online)
	- [Publication](https://doi.org/10.1093/nar/gkac813) : Citations: 44.0
	- [Link](https://idrblab.org/drugmap/)
- **GDB-17**: The GDB-17 database represents a comprehensive collection of hypothetical organic molecules for use in computational chemistry and drug discovery.
	- [Publication](https://doi.org/10.1021/ci300415d) : Citations: 874.0
	- [Link](https://gdb.unibe.ch/downloads/)
- **PubChem**: The world's largest collection of freely accessible chemical information, offering chemical and physical properties, biological activities, safety and toxicity information, patents, literature citations, and more.
	- [Publication](https://doi.org/10.1093/nar/gkac956) : Citations: 727.0
	- [Link](https://pubchem.ncbi.nlm.nih.gov/)

## Chemical Reactions
- **Open Reaction Database**: open access chemical reaction database to support machine learning and related efforts in reaction prediction, chemical synthesis planning, and experiment design
	- [Code](https://github.com/open-reaction-database/ord-schema) : Last updated in June 2024, 0 months ago
	- [Publication](https://doi.org/10.1021/jacs.1c09820) : Citations: 128.0
	- [Link](https://docs.open-reaction-database.org/en/latest/)

## Commercial Libraries
- **eMolecules**: commercially-available compounds
	- [Link](https://www.emolecules.com/)
- **MCULE**: Commercial compound collection (online)
	- [Link](https://mcule.com/database/)
- **Molport**: commercially-available compounds
	- [Link](https://www.molport.com/shop/index)
- **Zinc**: a free database of commercially-available compounds
	- [Publication](https://doi.org/10.1021/acs.jcim.0c00675) : Citations: 357.0
	- [Link](http://zinc15.docking.org/)
- **ZINC Express**: A Virtual Assistant for Purchasing Compounds Annotated in the ZINC Database (online)
	- [Publication](https://doi.org/10.1021/acs.jcim.0c00675) : Citations: 357.0
	- [Link](https://zincexpress.mml.unc.edu/)
- **ZINC20**: 
	- [Publication](https://doi.org/10.1021/acs.jcim.0c00675) : Citations: 357.0
	- [Link](http://zinc20.docking.org/)

## Docking and Scoring Function Benchmarks
- **BigBind and BayesBind**: 
	- [Code](https://github.com/molecularmodelinglab/bigbind) : Last updated in April 2024, 2 months ago
	- [Publication](https://doi.org/10.1021/acs.jcim.3c01211) : Citations: 2.0
- **CASF**: CASF offers benchmark packages for the assessment of scoring functions, with versions CASF-2016, CASF-2013, and CASF-2007 available for download upon registration.
	- [Publication](https://doi.org/10.1021/acs.jcim.8b00545) : Citations: 378.0
	- [Link](http://www.pdbbind.org.cn/casf.php)
- **DEKOIS**: Provides DEKOIS 2.0, a library of benchmark data sets for objective in silico screening.
	- [Publication](https://doi.org/10.1021/ci2001549) : Citations: 65.0
	- [Link](http://www.pharmchem.uni-tuebingen.de/dekois/)
- **Disco**: A default benchmark containing 95 targets selected from the DUD-E Targets list for docking datasets generation.
	- [Publication](http://disco.csb.pitt.edu/Targets_top1.php?ligands) : Citations: N/A
- **Lit-PCBA**: A dataset for virtual screening and machine learning with 15 target sets, 7761 actives, and 382674 unique inactives selected from high-confidence PubChem Bioassay data.
	- [Publication](https://doi.org/10.1021/acs.jcim.0c00155) : Citations: 96.0
	- [Link](https://drugdesign.unistra.fr/LIT-PCBA/index.html)
- **MF-PCBA**: A repository containing code for downloading, filtering, and assembling the MF-PCBA datasets for machine learning in drug discovery.
	- [Code](https://github.com/davidbuterez/mf-pcba) : Last updated in May 2023, 12 months ago
	- [Publication](https://doi.org/10.1021/acs.jcim.2c01569) : Citations: 7.0
- **MUDB-DecoyMaker2.0**: A Python GUI application to generate maximal unbiased benchmarking sets data sets for virtual drug screening.
	- [Code](https://github.com/jwxia2014/MUBD-DecoyMaker2.0) : Last updated in October 2023, 7 months ago
	- [Publication](https://doi.org/10.1002/minf.201900151) : Citations: 10.0
- **TocoDDB**: 
	- [Publication](https://doi.org/10.1021/acs.jmedchem.3c00801) : Citations: 2.0
	- [Link](http://cadd.zju.edu.cn/tocodecoy/)
- **TocoDecoy**: A new approach to design unbiased datasets for training and benchmarking machine-learning scoring functions.
	- [Code](https://github.com/5AGE-zhang/TocoDecoy) : Last updated in April 2022, 26 months ago

## Drugs
- **DrugCentral**: DrugCentral provides information on active ingredients in medicinal substances, offering access to drug labels, FDA and EMA datasets, pharmacological action, and more.
	- [Publication](https://doi.org/10.1007/s10822-023-00529-x) : Citations: 1.0
	- [Link](https://drugcentral.org/)

## GPCRs
- **GproteinDb**: The G protein database (online)
	- [Publication](https://dx.doi.org/10.1093/nar/gkx1109) : Citations: 395.0
	- [Link](https://gproteindb.org/)

## Kinases
- **CORAL**: Clear and customizable visualization of human kinome data (Kinase) (online)
	- [Code](https://github.com/dphansti/CORAL) : Last updated in August 2021, 34 months ago
	- [Publication](http://dx.doi.org/10.1016/j.cels.2018.07.001) : Citations: 118.0
	- [Link](http://phanstiel-lab.med.unc.edu/CORAL/)
- **KIBA**: An integrated drug–target bioactivity matrix across 52498 chemical compounds and 467 kinase targets (dataset)
	- [Publication](https://doi.org/doi:10.1021/ci400709d) : Citations: 294.0
	- [Link](https://researchportal.helsinki.fi/fi/datasets/kiba-a-benchmark-dataset-for-drug-target-prediction)
- **Kinase Atlas**: contains solvent mapping results for all kinases in the Protein Data Bank. You can search this database by PDB ID or by UniProt accession number (online)
	- [Publication](https://doi.org/10.1021/acs.jmedchem.9b00089) : Citations: 51.0
	- [Link](https://kinase-atlas.bu.edu/)
- **KinaseMD**: kinase mutations and drug response database (online)
	- [Publication](https://doi.org/10.1093/nar/gkaa945) : Citations: 41.0
	- [Link](https://bioinfo.uth.edu/kmd/)
- **KinBase**: kinase (online)
	- [Link](http://kinase.com/web/current/kinbase/)
- **KinCoRe**: A web resource for protein kinase sequence, structure and phylogeny (online)
	- [Publication](https://doi.org/10.1101/2023.07.21.550125) : Citations: 11.0
	- [Link](http://dunbrack.fccc.edu/kincore/)
- **KinMap**: a web-based tool for interactive navigation through human kinome data (kinase) (online)
	- [Publication](https://doi.org/10.1186/s12859-016-1433-7) : Citations: 225.0
	- [Link](http://www.kinhub.org/kinmap/)
- **Kinome Render**: look at kinase family (standalone)
	- [Link](http://biophys.umontreal.ca/nrg/resources.html)
- **KLIFS**: Kinase-Ligand Interaction Fingerprints and Structures database is a database that revolves around the protein structure of catalytic kinase domains and the way kinase inhibitors can interact with them (online database)
	- [Publication](https://doi.org/10.1093/nar/gkaa895) : Citations: 80.0
	- [Link](https://klifs.vu-compmedchem.nl/)
- **ProfKin**: A Comprehensive Web Server for Structure-based Kinase Selectivity Profiling (online)
	- [Publication](https://doi.org/10.1016/j.ejmech.2021.113772) : Citations: 8.0
	- [Link](http://www.lilab-ecust.cn/profkin/)

## Machine Learning Benchmarks
- **Therapeutics Data Commons**: Therapeutics Data Commons is a resource to access and evaluate AI methods, supporting the development of AI methods, with a strong bent towards establishing the foundation of which AI methods are most suitable for drug discovery applications and why.
	- [Link](https://tdcommons.ai/)

## Make on Demand Libraries
- **Enamine REAL**: 
	- [Link](https://enamine.net/compound-collections/real-compounds/real-database)
- **FreedomSpace**: 
	- [Link](https://chem-space.com/compounds/freedom-space)

## Molecular Dynamics
- **ATLAS**: Resource for exploring molecular dynamics simulations, offering insights into the dynamics of proteins and other biomolecules.
	- [Publication](https://doi.org/10.1093/nar/gkad1084) : Citations: 5.0
	- [Link](https://www.dsimb.inserm.fr/ATLAS)
- **MISATO**: machine learning dataset of protein–ligand complexes for structure-based drug discovery
	- [Publication](http://dx.doi.org/10.1038/s43588-024-00627-2) : Citations: 1.0
	- [Link](https://zenodo.org/records/7711953)
- **PLAS-5k**: Dataset of Protein-Ligand Affinities from Molecular Dynamics for Machine Learning Applications (5000 protein-ligand complexes) (dataset).
	- [Link](https://hai.iiit.ac.in/datasets.html)

## Natural Products
- **BIOFACQUIM**: novel compounds databse with natural products isolated and charcterized in Mexico (online)
	- [Link](https://www.difacquim.com/d-tools/)
- **COCONUT**: COCONUT hosts one of the largest and best annotated natural product databases, freely available and undergoing continuous curation and improvement.
	- [Publication](https://doi.org/10.1186%2Fs13321-020-00478-9) : Citations: 243.0
	- [Link](https://coconut.naturalproducts.net/)
- **HIT 2.0**: an enhanced platform for Herbal Ingredients' Targets (online)
	- [Publication](https://doi.org/10.1093/nar/gkq1165) : Citations: 246.0
	- [Link](http://hit2.badd-cao.net/)
- **iSMART**: an integrated cloud computing web server for traditional Chinese medicine for online virtual screening, de novo evolution and drug design (online)
	- [Link](http://ismart.cmu.edu.tw/)
- **LOTUS**: LOTUS is a significant, well-annotated resource for natural products, offering a user-friendly experience including structural search, taxonomy-oriented queries, and exports.
	- [Publication](https://doi.org/10.7554/eLife.70780) : Citations: 102.0
	- [Link](https://lotus.naturalproducts.net/)
- **Natural product subsets (DIFACQUIM)**: Natural product datasets (diverse with Python code to generate diversity) (dataset)
	- [Code](https://github.com/DIFACQUIM/Natural-products-subsets-generation) : Last updated in October 2023, 7 months ago
	- [Publication](https://doi.org/10.1016/j.ailsci.2023.100066) : Citations: 2.0
- **NPASS**: Natural product activity and species source database
	- [Publication](https://doi.org/10.1093/nar/gkac1069) : Citations: 12.0
	- [Link](https://bidd.group/NPASS/)
- **npatlas**: The Natural Products Atlas 2.0: A Database of Microbially-Derived Natural Products (online)
	- [Link](https://www.npatlas.org/)
- **Nubbe**: a natural products database from the biodiversity of Brazil (online)
	- [Link](http://nubbe.iq.unesp.br/portal/nubbe-search.html)
- **PeruNPDB**: the Peruvian Natural Products Database for in silico drug screening (database)
	- [Link](https://perunpdb.com.pe/)
- **Super Natural II**: a database of natural products (online)
	- [Publication](https://doi.org/10.1093/nar/gkac1008.) : Citations: 0.0
	- [Link](http://bioinf-applied.charite.de/supernatural_new/index.php)
- **SymMap**: SymMap integrates traditional Chinese medicine with modern medicine through molecular mechanisms and symptom mapping, offering information on herbs, ingredients, targets, symptoms, and diseases.
	- [Publication](https://doi.org/10.1186/s13321-020-00478-9) : Citations: 243.0
	- [Link](http://www.symmap.org/)
- **TCM Database@Taiwan**: the world's largest traditional Chinese medicine database for drug screening in silico (online)
	- [Link](http://tcm.cmu.edu.tw/)
- **TCMSID**: a simplified integrated database for drug discovery from traditional chinese medicine (online)
	- [Publication](https://doi.org/10.1186/s13321-022-00670-z) : Citations: 6.0
	- [Link](https://tcm.scbdd.com/)
- **VIETHERB**: A Database for Vietnamese Herbal Species - Natural products (online)
	- [Link](http://vietherb.com.vn/)

## PROTACs
- **PROTAC-DB**: an online database of PROTACs (online)
	- [Link](http://cadd.zju.edu.cn/protacdb/)
- **PROTAC-DB 2.0**: an updated database of PROTACs (online)
	- [Link](http://cadd.zju.edu.cn/protacdb/)

## Properties
- **MoleculeNet**: - Selection of data sets of molecules (SMILES) and physicochemical properties
	- [Code](https://github.com/GLambard/Molecules_Dataset_Collection?tab=readme-ov-file) : Last updated in June 2018, 72 months ago
	- [Publication](https://doi.org/10.48550/arXiv.1703.00564) : Citations: 0.0

## Protein Sequences
- **OpenProteinSet**: OpenProteinSet is an open-source corpus of more than 16 million Multiple Sequence Alignments (MSAs), associated structural homologs from the Protein Data Bank, and AlphaFold2 protein structure predictions. It's designed to be broadly useful as training and validation data for tasks focused on protein structure, function, and design, as well as large-scale multimodal machine learning research.
	- [Publication](https://doi.org/10.48550/arXiv.2308.05326) : Citations: 0.0
	- [Link](https://arxiv.org/abs/2308.05326)

## Protein Structures
- **PDB**: The RCSB Protein Data Bank provides access to experimentally determined 3D structures from the PDB archive and computed structure models, offering tools for exploration, visualization, and analysis.
	- [Publication](https://doi.org/10.1093/nar/28.1.235) : Citations: 28739.0
	- [Link](https://www.rcsb.org/)
- **PDB-redo**: Offers optimized versions of PDB entries through an automated procedure that refines, rebuilds, and validates structural models, combining popular crystallographic software with specially developed tools.
	- [Publication](https://doi.org/10.1107/s0907444911054515) : Citations: 185.0
	- [Link](https://pdb-redo.eu/)

## Protein Targets
- **OpenTargets**: A partnership to transform drug discovery through the systematic identification and prioritisation of targets
	- [Publication](https://doi.org/10.1093/nar/gkw1055) : Citations: 347.0
	- [Link](https://www.opentargets.org/)

## Protein-Ligand Interaction
- **BindingDB**: BindingDB contains data for over 1.2 million compounds and 9.2k targets, supporting research, education, and practice in drug discovery, pharmacology, and related fields.
	- [Publication](https://doi.org/10.1093/nar/gkv1072) : Citations: 978.0
	- [Link](https://www.bindingdb.org/bind/index.jsp)
- **BindingNet**: BindingNet is a dataset for analyzing protein-ligand interactions, containing modeled poses for compounds similar to the crystal ligands found in PDBbind, along with corresponding activities from ChEMBL.
	- [Code](https://github.com/hnlab/BindingNet) : Last updated in January 2024, 5 months ago
	- [Publication](https://doi.org/10.1021/acs.jcim.3c01170) : Citations: 1.0
	- [Link](http://bindingnet.huanglab.org.cn/)
- **BioLip2**: BioLiP is a semi-manually curated database for high-quality, biologically relevant ligand-protein binding interactions, aiming to serve the needs of ligand-protein docking, virtual ligand screening, and protein function annotation.
	- [Publication](https://doi.org/10.1093/nar/gkad630) : Citations: 12.0
	- [Link](https://zhanggroup.org/BioLiP/index.cgi)
- **Leak Proof PDBBind**: This work presents a cleaned PDBBind dataset of non-covalent binders, reorganized to avoid data leakage, allowing for more generalizable binding affinity prediction.
	- [Code](https://github.com/THGLab/LP-PDBBind) : Last updated in January 2024, 4 months ago
	- [Publication](https://doi.org/10.48550/arXiv.2308.09639) : Citations: 0.0
	- [Link](https://github.com/THGLab/LP-PDBBind)
- **PDBBind**: 
	- [Publication](https://doi.org/10.1021/acs.accounts.6b00491) : Citations: 266.0
	- [Link](http://pdbbind.org.cn/)
- **PDBscreen**: PDBscreen provides a dataset with multiple data augmentation strategies suitable for training protein-ligand interaction prediction methods.
	- [Link](https://zenodo.org/records/8049380)
- **PLAS-20K**: 
	- [Publication](https://doi.org/10.26434/chemrxiv-2023-mg07d) : Citations: 0.0
	- [Link](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/64cca66569bfb8925a5514c5/original/plas-20k-extended-dataset-of-protein-ligand-affinities-from-md-simulations-for-machine-learning-applications.pdf)
- **PDBBind+**: 
	- [Link](https://www.pdbbind-plus.org.cn/)

## Toxicity
- **DrugMatrix/ToxFX**: one of the world’s largest toxicogenomic reference resources
	- [Link](https://ntp.niehs.nih.gov/data/drugmatrix)
- **hergcentral**: 
	- [Publication](https://doi.org/10.1089%2Fadt.2011.0425) : Citations: 37.0
	- [Link](http://www.hergcentral.org/)
### hERG
- **hERGdb**: web-based cardiotoxicity prediction
	- [Webserver](https://drugdesign.riken.jp/hERGdb/)
