---
icon: material/timeline-question
---


## **General**
- **AlphaFold**: Protein Structure Database (predicted human 3D proteins, July 2021)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:24347-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038/s41586-021-03819-2) [![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://alphafold.ebi.ac.uk/) 
- **AlphaFold-Multimer**: a model that can predict the structure of multi-chain protein complexes with increased accuracy (standalone)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:1-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1371/journal.pcbi.1012253) [![Link](https://img.shields.io/badge/Link-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](https://colab.research.google.com/github/deepmind/alphafold/blob/main/notebooks/AlphaFold.ipynb) 
- **ColabFold**: making protein folding accessible to all (standalone)  
		[![Code](https://img.shields.io/github/stars/sokrypton/ColabFold?style=for-the-badge&logo=github)](https://github.com/sokrypton/ColabFold) [![Last Commit](https://img.shields.io/github/last-commit/sokrypton/ColabFold?style=for-the-badge&logo=github)](https://github.com/sokrypton/ColabFold) [![Publication](https://img.shields.io/badge/Publication-Citations:4994-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038/s41592-022-01488-1) 
- **DMFold**: DMFold standalone package is an integrated program of DeepMSA2 and AlphaFold2 for protein monomer and protein complex structure prediction  
	[![Publication](https://img.shields.io/badge/Publication-Citations:N/A-blue?style=for-the-badge&logo=bookstack)](s41592-023-02130-4) [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://zhanggroup.org/DMFold/download/) 
- **DMFold**: A tool that integrates large genomic and metagenomics sequence databases for improved protein structure prediction.  
	[![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://zhanggroup.org/DMFold/download/) 
- **ESM**: Evolutionary Scale Modeling to predict protein 3D structure (standalone) (online)  
		[![Code](https://img.shields.io/github/stars/facebookresearch/esm?style=for-the-badge&logo=github)](https://github.com/facebookresearch/esm) [![Last Commit](https://img.shields.io/github/last-commit/facebookresearch/esm?style=for-the-badge&logo=github)](https://github.com/facebookresearch/esm) [![Publication](https://img.shields.io/badge/Publication-Citations:9-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1186/s12859-022-04628-8) 
- **ESMFold**: ESM Metagenomic Atlas contains several millions of predicted protein structures (can be used via ChimeraX) (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:1386-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1126/science.ade2574) [![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://esmatlas.com/about) 
- **Evo**: A long-context foundation model that generalizes across the central dogma of biology: DNA, RNA, and proteins.  
		[![Code](https://img.shields.io/github/stars/evo-design/evo?style=for-the-badge&logo=github)](https://github.com/evo-design/evo) [![Last Commit](https://img.shields.io/github/last-commit/evo-design/evo?style=for-the-badge&logo=github)](https://github.com/evo-design/evo) [![Publication](https://img.shields.io/badge/Publication-Citations:98-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038/s41588-022-01065-4) 
- **McGuffin Group Web Servers**: This link points to the home page of the McGuffin Group Web Servers at the University of Reading, which provides various bioinformatics tools, although specific details about the tools were not provided.  
	[![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://www.reading.ac.uk/bioinf/index.html) 
- **Openfold**: A faithful PyTorch reproduction of DeepMind's AlphaFold2 (standalone)  
		[![Code](https://img.shields.io/github/stars/aqlaboratory/openfold?style=for-the-badge&logo=github)](https://github.com/aqlaboratory/openfold) [![Last Commit](https://img.shields.io/github/last-commit/aqlaboratory/openfold?style=for-the-badge&logo=github)](https://github.com/aqlaboratory/openfold) [![Publication](https://img.shields.io/badge/Publication-Citations:35-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038/s41592-024-02272-z) 
- **Raptor-X**: RaptorX-Single: exploring the advantage of single sequence based protein structure prediction  
		[![Code](https://img.shields.io/github/stars/AndersJing/RaptorX-Single?style=for-the-badge&logo=github)](https://github.com/AndersJing/RaptorX-Single) [![Last Commit](https://img.shields.io/github/last-commit/AndersJing/RaptorX-Single?style=for-the-badge&logo=github)](https://github.com/AndersJing/RaptorX-Single) [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.5281/zenodo.7351378) 
- **Robetta**: Robetta offers structure prediction using deep learning methods like RoseTTAFold and TrRosetta. It allows for custom sequence alignments, constraints, local fragments for homology modeling, and can model multi-chain complexes.  
	[![Publication](https://img.shields.io/badge/Publication-Citations:1649-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093%2Fnar%2Fgkh468) [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://robetta.bakerlab.org/) 
- **RoseTTAFold**: This package contains deep learning models and scripts for RoseTTAFold, an accurate method for protein structure prediction that includes a 3-track network. It's capable of modeling multi-chain complexes and provides an option for large scale sampling.  
		[![Code](https://img.shields.io/github/stars/RosettaCommons/RoseTTAFold?style=for-the-badge&logo=github)](https://github.com/RosettaCommons/RoseTTAFold) [![Last Commit](https://img.shields.io/github/last-commit/RosettaCommons/RoseTTAFold?style=for-the-badge&logo=github)](https://github.com/RosettaCommons/RoseTTAFold) [![Publication](https://img.shields.io/badge/Publication-Citations:3516-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1126/science.abj8754) 

## **Quaternary structure**
- **AlphaFold-multimer**: The specific details about AlphaFold-multimer could not be directly retrieved, but it extends AlphaFold2's capabilities to predict structures of protein complexes (multimers).  
		[![Code](https://img.shields.io/github/stars/google-deepmind/alphafold?style=for-the-badge&logo=github)](https://github.com/google-deepmind/alphafold) [![Last Commit](https://img.shields.io/github/last-commit/google-deepmind/alphafold?style=for-the-badge&logo=github)](https://github.com/google-deepmind/alphafold) [![Publication](https://img.shields.io/badge/Publication-Citations:24347-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038/s41586-021-03819-2) 
- **CombFold**: a combinatorial and hierarchical assembly algorithm combined with AlphaFold2 for predicting structures of large protein assemblies  
	[![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://lnkd.in/gRVdfaZV) 
- **DeepComplex**: This document has moved, and direct content was not provided in the data fetched, but DeepComplex is aimed at predicting quaternary protein structures.  
	[![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://tulip.rnet.missouri.edu/deepcomplex/web_index.html) 

## **Structure Prediction with Ligand**
- **Chai**: Chai-1: Decoding the molecular interactions of life  
		[![Code](https://img.shields.io/github/stars/chaidiscovery/chai-lab?style=for-the-badge&logo=github)](https://github.com/chaidiscovery/chai-lab) [![Last Commit](https://img.shields.io/github/last-commit/chaidiscovery/chai-lab?style=for-the-badge&logo=github)](https://github.com/chaidiscovery/chai-lab) [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/hesc/9780198830108.003.0048) 
- **DiffusionProteinLigand**: End-to-end proteinâ€“ligand complex structure generation with diffusion-based generative models (standalone).  
		[![Code](https://img.shields.io/github/stars/shuyana/DiffusionProteinLigand?style=for-the-badge&logo=github)](https://github.com/shuyana/DiffusionProteinLigand) [![Last Commit](https://img.shields.io/github/last-commit/shuyana/DiffusionProteinLigand?style=for-the-badge&logo=github)](https://github.com/shuyana/DiffusionProteinLigand) [![Publication](https://img.shields.io/badge/Publication-Citations:16-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1186/s12859-023-05354-5) 
- **DynamicBind**: DynamicBind recovers ligand-specific conformations from unbound protein structures (e.g. AF2-predicted structures), promoting efficient transitions between different equilibrium states.  
		[![Code](https://img.shields.io/github/stars/luwei0917/DynamicBind?style=for-the-badge&logo=github)](https://github.com/luwei0917/DynamicBind) [![Last Commit](https://img.shields.io/github/last-commit/luwei0917/DynamicBind?style=for-the-badge&logo=github)](https://github.com/luwei0917/DynamicBind) [![Publication](https://img.shields.io/badge/Publication-Citations:21-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038/s41467-024-45461-2) 
- **NeuralPlexer**: a deep generative model to jointly predict protein-ligand complex 3D structures and beyond.  
		[![Code](https://img.shields.io/github/stars/zrqiao/NeuralPLexer?style=for-the-badge&logo=github)](https://github.com/zrqiao/NeuralPLexer) [![Last Commit](https://img.shields.io/github/last-commit/zrqiao/NeuralPLexer?style=for-the-badge&logo=github)](https://github.com/zrqiao/NeuralPLexer) [![Publication](https://img.shields.io/badge/Publication-Citations:27-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038/s42256-024-00792-z) 
- **RoseTTAFold_AllAtom**: biomolecular structure prediction neural network that can predict a broad range of biomolecular assemblies including proteins, nucleic acids, small molecules, covalent modifications and metals as outlined in the RFAA paper.  
		[![Code](https://img.shields.io/github/stars/AaronFeller/RoseTTAFold-All-Atom?style=for-the-badge&logo=github)](https://github.com/AaronFeller/RoseTTAFold-All-Atom/blob/main/README.md) [![Last Commit](https://img.shields.io/github/last-commit/AaronFeller/RoseTTAFold-All-Atom?style=for-the-badge&logo=github)](https://github.com/AaronFeller/RoseTTAFold-All-Atom/blob/main/README.md) [![Publication](https://img.shields.io/badge/Publication-Citations:134-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1126/science.adl2528) 
- **Umol**: Umol is designed for protein-ligand structure prediction, representing the protein with a multiple sequence alignment and the ligand as a SMILES string, with versions utilizing protein pocket information recommended.  
		[![Code](https://img.shields.io/github/stars/patrickbryant1/Umol?style=for-the-badge&logo=github)](https://github.com/patrickbryant1/Umol) [![Last Commit](https://img.shields.io/github/last-commit/patrickbryant1/Umol?style=for-the-badge&logo=github)](https://github.com/patrickbryant1/Umol) [![Publication](https://img.shields.io/badge/Publication-Citations:8-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038/s41467-024-48837-6) 

## **Trans-membrane Proteins**
- **MemBrain**: The site provides resources related to membrane protein prediction but specific details about the MemBrain tool were not provided.  
	[![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://www.csbio.sjtu.edu.cn/bioinf/MemBrain/) 
- **membraneFold**: This resource is intended for predicting membrane protein structures, but specific details were not provided.  
	[![Publication](https://img.shields.io/badge/Publication-Citations:8-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/oso/9780199634972.003.0005) [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://ku.biolib.com/MembraneFold/) 
- **PredMP**: The website's specific content was not retrievable due to request issues, but PredMP is designed for predicting membrane protein types and orientations.  
	[![Link](https://img.shields.io/badge/Link-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](http://www.predmp.com/) 
- **RosettaGPCR**: This repository contains methods for generating models of G protein-coupled receptors (GPCRs) using Rosetta, including a database of templates updated through June 2020.  
		[![Code](https://img.shields.io/github/stars/benderb1/rosettagpcr?style=for-the-badge&logo=github)](https://github.com/benderb1/rosettagpcr) [![Last Commit](https://img.shields.io/github/last-commit/benderb1/rosettagpcr?style=for-the-badge&logo=github)](https://github.com/benderb1/rosettagpcr) [![Publication](https://img.shields.io/badge/Publication-Citations:7-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1007/978-1-60327-317-6_7) 
