---
icon: material/set-all
---

- **3D-Garden**: A system for modelling protein-protein complexes based on conformational refinement of ensembles generated with the marching cubes algorithm (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:53-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btn093) [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://www.sbg.bio.ic.ac.uk/~3dgarden/) 
- **AREA-AFFINITY**: Server for Machine Learning-Based Prediction of Protein-Protein and Antibody-Protein Antigen Binding Affinities (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:8-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.2c01499) [![Link](https://img.shields.io/badge/Link-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](https://affinity.cuhk.edu.cn/) 
- **Arpeggio**: A webserver for calculating interatomic interactions in protein structures (online & standalone)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:363-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1016/j.jmb.2016.12.004) [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://biosig.unimelb.edu.au/arpeggioweb/) 
- **BAlaS**: fast, interactive and accessible computational alanine-scanning using BudeAlaScan (online)  
		[![Code](https://img.shields.io/github/stars/wells-wood-research/balas?style=for-the-badge&logo=github)](https://github.com/wells-wood-research/balas) [![Last Commit](https://img.shields.io/github/last-commit/wells-wood-research/balas?style=for-the-badge&logo=github)](https://github.com/wells-wood-research/balas) [![Publication](https://img.shields.io/badge/Publication-Citations:48-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btaa026) 
- **Cluspro**: protein docking (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:2153-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038%2Fnprot.2016.169) [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://cluspro.bu.edu/login.php) 
- **ClusPro FMFT-SAXS**: Ultra-fast Filtering Using Small-Angle X-ray Scattering Data in Protein Docking (online and standalone)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:2153-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038%2Fnprot.2016.169) [![Link](https://img.shields.io/badge/Link-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](https://beta.cluspro.org/) 
- **ClusPro TBM**: Toward combining template-based methods with free docking (protein-protein docking) (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:2153-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038%2Fnprot.2016.169) [![Link](https://img.shields.io/badge/Link-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](https://tbm.cluspro.org/) 
- **ClusPro-DC**: discrimination between Xray and biological dimers (online requires login)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:2153-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038%2Fnprot.2016.169) [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://cluspro.bu.edu/) 
- **CoDockPP**: A Multistage Approach for Global and Site-Specific Protein-Protein Docking (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:32-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.9b00445) [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://codockpp.schanglab.org.cn/) 
- **DeepComplex**: A Web Server of Predicting Protein Complex Structures by Deep Learning Inter-chain Contact Prediction and Distance-Based Modelling (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:15-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.3389/fmolb.2021.716973) [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://tulip.rnet.missouri.edu/deepcomplex/web_index.html) 
- **DisVis**: visualizes the accessible interaction space (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/bt) [![Webserver](https://img.shields.io/badge/Webserver-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](http://milou.science.uu.nl/cgi/services/DISVIS/disvis/) 
- **DOCK/PIERR**: Protein Docking Server (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:26-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1007/978-1-4939-0366-5_14) [![Webserver](https://img.shields.io/badge/Webserver-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](http://clsb.ices.utexas.edu/web/dock.html) 
- **Dockground**: A comprehensive data resource for modeling of protein complexes, datasets, decoys... (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:11-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1002/pro.4481) [![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://dockground.compbio.ku.edu/) 
- **DOCKSCORE**: rank docked poses (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:N/A-blue?style=for-the-badge&logo=bookstack)](127) [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://caps.ncbs.res.in/dockscore/) 
- **DockStar**: Modelling of multimolecular protein complexes (online)  
	[![Webserver](https://img.shields.io/badge/Webserver-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](http://bioinfo3d.cs.tau.ac.il/DockStar/) 
- **DockSurf**: A molecular modeling software for the prediction of protein/surface adhesion (eg. gold surfaces) (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.3c00569) [![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://bioserv.rpbs.univ-paris-diderot.fr/services/DockSurf/) 
- **DOVE**: convolutional deep neural network-based approach named DOcking decoy selection with Voxel-based deep neural nEtwork (DOVE) for evaluating protein docking models (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:88-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btz870) [![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://kiharalab.org/dove/) 
- **EROS-DOCK**: proteinâ€“protein docking using exhaustive branch-and-bound rotational search (standalone)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:9-blue?style=for-the-badge&logo=bookstack)](doi:10.1093/bioinformatics/btz434) [![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://erosdock.loria.fr/) 
- **ezCADD**: a Rapid 2D/3D Visualization Enabled Web Modeling Environment for Democratizing Computer-Aided Drug Design (small molecule docking, protein docking, 2D/3D interaction viz, de novo lead optimization, pocket search and polypharmacology) online  
	[![Publication](https://img.shields.io/badge/Publication-Citations:56-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.8b00633) [![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://www.dxulab.org/software) 
- **FiberDock, FireDock, FlexDock, PatchDock**: FiberDock: a web server for flexible induced-fit backbone refinement in molecular docking  
	[![Publication](https://img.shields.io/badge/Publication-Citations:86-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/nar/gkq373) [![Webserver](https://img.shields.io/badge/Webserver-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](http://bioinfo3d.cs.tau.ac.il/wk/index.php/Servers_%26_Software) 
- **FiltRest3D**: filter predicted models (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:44-blue?style=for-the-badge&logo=bookstack)](doi:10.1371/journal.pcbi.1008309) [![Webserver](https://img.shields.io/badge/Webserver-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](http://filtrest3d.genesilico.pl/filtrest3d/help.html) 
- **FoldDock**: simultaneous folding and docking protocol FoldDock and AlphaFold (standalone)  
	[![Code](https://img.shields.io/badge/Code-Repository-blue?style=for-the-badge)](https://gitlab.com/ElofssonLab/FoldDock/) [![Publication](https://img.shields.io/badge/Publication-Citations:505-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038/s41467-022-28865-w) 
- **FRODOCK**: Fast Rotational DOCKing (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:127-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btw141) [![Webserver](https://img.shields.io/badge/Webserver-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](http://frodock.chaconlab.org/) 
- **GalaxyDomDock**: An Ab Initio Domain-domain Docking Web Server for Multi-domain Protein Structure Prediction (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:6-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1016/j.jmb.2022.167508) [![Webserver](https://img.shields.io/badge/Webserver-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](http://galaxy.seoklab.org/domdock) 
- **GalaxyHomomer**: protein homo-oligomer structure prediction from a monomer sequence or structure (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:109-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/nar/gkx246) [![Link](https://img.shields.io/badge/Link-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](http://galaxy.seoklab.org/cgi-bin/submit.cgi?type=HOMOMER) 
- **GalaxyRefineComplex**: Protein-protein complex structures generated by protein-protein docking methods can be refined (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:103-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038%2Fsrep32153) [![Webserver](https://img.shields.io/badge/Webserver-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](http://galaxy.seoklab.org/cgi-bin/submit.cgi?type=COMPLEX) 
- **GalaxyTongDock**: Symmetric and asymmetric ab initio protein-protein docking web server with improved energy parameters (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:662-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/nar/gks493) [![Link](https://img.shields.io/badge/Link-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](http://galaxy.seoklab.org/tongdock) 
- **GRAMM**: Protein-Protein Docking Web Server (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:16-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1007/978-1-0716-3441-7_5) [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://gramm.compbio.ku.edu/) 
- **GraphPPIS**: Structure-aware protein-protein interaction site prediction using deep graph convolutional network (online), standalone on github  
	[![Publication](https://img.shields.io/badge/Publication-Citations:93-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btab643) [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://biomed.nscc-gz.cn/apps/GraphPPIS) 
- **GWIDD**: genome wide docking database (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:14-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1186/1479-7364-6-7) [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://gwidd.compbio.ku.edu/) 
- **Haddock**: High Ambiguity Driven protein-protein DOCKing (online requires login)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:12-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.21769%2FBioProtoc.3793) [![Webserver](https://img.shields.io/badge/Webserver-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](http://haddock.chem.uu.nl/) 
- **HawkDock**: to predict and analyze the protein-protein complex based on computational docking and MM/GBSA (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:388-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/nar/gkz397) [![Link](https://img.shields.io/badge/Link-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](http://cadd.zju.edu.cn/hawkdock/) 
- **HDOCK**: hybrid proteinâ€“protein and proteinâ€“DNA/RNA docking, Pushing the accuracy limit of shape complementarity for protein-protein docking (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:792-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093%2Fnar%2Fgkx407) [![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://hdock.phys.hust.edu.cn/) 
- **HMI-PRED**: A Web Server for Structural Prediction of Host-Microbe Interactions Based on Interface Mimicry (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:2-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btac633) [![Webserver](https://img.shields.io/badge/Webserver-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](https://interactome.ku.edu.tr/hmi/) 
- **InterEvDock2**: protein docking using evolutionary and biological information from homology models and multimeric inputs (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:48-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093%2Fnar%2Fgky377) [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://bioserv.rpbs.univ-paris-diderot.fr/services/InterEvDock2/) 
- **InterEvDock3**: a combined template-based and free docking server with increased performance through explicit modeling of complex homologs and integration of covariation-based contact maps (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:20-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093%2Fnar%2Fgkab358) [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://bioserv.rpbs.univ-paris-diderot.fr/services/InterEvDock3/) 
- **InterPred**: A pipeline to identify and model protein-protein interactions (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:34-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1002/prot.25280) [![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://wallnerlab.org/InterPred) 
- **INTERSPIA**: Explore the Dynamics of Protein-Protein Interactions Among Multiple Species (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:9-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/nar/gky378) [![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://bioinfo.konkuk.ac.kr/INTERSPIA/) 
- **IRaPPA**: pyDockRescoring Server (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:37-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btx068) [![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://life.bsc.es/pid/pydockrescoring/) 
- **LightDock**: Artificial Intelligence-powered modeling of macromolecular interactions (macromolecular docking framework, online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:86-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btx555) [![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://server.lightdock.org/) 
- **P3DOCK**: a protein-RNA docking webserver based on template-based and template-free docking (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:18-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btz478) [![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://www.rnabinding.com/P3DOCK/P3DOCK.html) 
- **PIIMS**: Mutation Hotspot Scanning at the Protein-Protein Interface (online)  
	[![Link](https://img.shields.io/badge/Link-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](http://chemyang.ccnu.edu.cn/ccb/server/PIIMS/index.php) 
- **PPI-Affinity**: Prediction and Optimization of Protein-Peptide and Protein-Protein Binding Affinity (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:40-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jproteome.2c00020) [![Webserver](https://img.shields.io/badge/Webserver-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](https://protdcal.zmb.uni-due.de/PPIAffinity) 
- **PRODIGY**: PROtein binDIng enerGY prediction (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:832-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btw514) [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://wenmr.science.uu.nl/prodigy/) 
- **PRODIGY-crystal**: classification of biological interfaces in protein complexes (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:29-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btz437) [![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://wenmr.science.uu.nl/prodigy/) 
- **pyDock**: pyDockWEB: a web server for rigid-body proteinâ€“protein docking using electrostatics and desolvation scoring  
	[![Publication](https://img.shields.io/badge/Publication-Citations:226-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btt262) [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://life.bsc.es/pid/pydock/) 
- **pyDockDNA**: A new web server for energy-based protein-DNA docking and scoring (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:13-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.3389/fmolb.2022.988996) [![Webserver](https://img.shields.io/badge/Webserver-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](https://model3dbio.csic.es/pydockdna) 
- **pyDockEneRes**: per-residue decomposition of protein-protein docking energy (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:18-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btz884) [![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://life.bsc.es/pid/pydockeneres) 
- **QASDOM**: Quality ASsessment of DOcking Models, protein-protein (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:10-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btx591) [![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://qasdom.eimb.ru/qasdom.html) 
- **RosettaDock**: Efficient Flexible Backbone Protein-Protein Docking for Challenging Targets (standalone and online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:483-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093%2Fnar%2Fgkn216) [![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://www.rosettacommons.org/) 
- **SPServer**: split-statistical potentials for the analysis of protein structures and protein-protein interactions (PPI) (online) (or http://sbi.upf.edu/spserver/)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:8-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1186/s12859-020-03770-5) [![Link](https://img.shields.io/badge/Link-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](https://sbi.upf.edu/spserver/) 
- **SwarmDock**: webservice for generating 3D structures of protein-protein complexes (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:1-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1007/978-1-0716-0708-4_11) [![Webserver](https://img.shields.io/badge/Webserver-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](http://bmm.crick.ac.uk/~SwarmDock/) 
- **webSDA**: Simulation of Diffusional Assosiation (SDA) (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/nar/gk) [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://websda.h-its.org/webSDA) 
- **ZDOCK**: docking (online)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:505-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1371/journal.pone.0024657) [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://zdock.umassmed.edu/) 

## **Quality Assessment**
- **DockQ**: A Quality Measure for Protein-Protein Docking Models (standalone)  
		[![Code](https://img.shields.io/github/stars/bjornwallner/DockQ?style=for-the-badge&logo=github)](https://github.com/bjornwallner/DockQ/) [![Last Commit](https://img.shields.io/github/last-commit/bjornwallner/DockQ?style=for-the-badge&logo=github)](https://github.com/bjornwallner/DockQ/) [![Publication](https://img.shields.io/badge/Publication-Citations:261-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1371/journal.pone.0161879) 
