



- **3D-Garden**: A system for modelling protein-protein complexes based on conformational refinement of ensembles generated with the marching cubes algorithm (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:54-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btn093) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/bioinformatics/btn093)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://www.sbg.bio.ic.ac.uk/~3dgarden/) 


- **AREA-AFFINITY**: Server for Machine Learning-Based Prediction of Protein-Protein and Antibody-Protein Antigen Binding Affinities (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:13-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.2c01499) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jcim.2c01499)



- **Arpeggio**: A webserver for calculating interatomic interactions in protein structures (online & standalone)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:379-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1016/j.jmb.2016.12.004) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.70-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1016/j.jmb.2016.12.004)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://biosig.unimelb.edu.au/arpeggioweb/) 


- **BAlaS**: fast, interactive and accessible computational alanine-scanning using BudeAlaScan (online)  
    [![Code](https://img.shields.io/github/stars/wells-wood-research/balas?style=for-the-badge&logo=github)](https://github.com/wells-wood-research/balas) 
    [![Last Commit](https://img.shields.io/github/last-commit/wells-wood-research/balas?style=for-the-badge&logo=github)](https://github.com/wells-wood-research/balas) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:55-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btaa026) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/bioinformatics/btaa026)



- **ClusPro FMFT-SAXS**: Ultra-fast Filtering Using Small-Angle X-ray Scattering Data in Protein Docking (online and standalone)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:2326-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038%2Fnprot.2016.169) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-13.10-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1038%2Fnprot.2016.169)



- **ClusPro TBM**: Toward combining template-based methods with free docking (protein-protein docking) (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:2326-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038%2Fnprot.2016.169) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-13.10-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1038%2Fnprot.2016.169)



- **ClusPro-DC**: discrimination between Xray and biological dimers (online requires login)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:2326-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038%2Fnprot.2016.169) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-13.10-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1038%2Fnprot.2016.169)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://cluspro.bu.edu/) 


- **Cluspro**: protein docking (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:2326-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038%2Fnprot.2016.169) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-13.10-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1038%2Fnprot.2016.169)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://cluspro.bu.edu/login.php) 


- **CoDockPP**: A Multistage Approach for Global and Site-Specific Protein-Protein Docking (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:35-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.9b00445) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jcim.9b00445)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://codockpp.schanglab.org.cn/) 


- **DOCK/PIERR**: Protein Docking Server (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:26-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1007/978-1-4939-0366-5_14) 

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://clsb.ices.utexas.edu/web/dock.html) 


- **DOCKSCORE**: rank docked poses (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](None) 

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://caps.ncbs.res.in/dockscore/) 


- **DOVE**: convolutional deep neural network-based approach named DOcking decoy selection with Voxel-based deep neural nEtwork (DOVE) for evaluating protein docking models (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:94-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btz870) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/bioinformatics/btz870)



- **DeepComplex**: A Web Server of Predicting Protein Complex Structures by Deep Learning Inter-chain Contact Prediction and Distance-Based Modelling (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:17-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.3389/fmolb.2021.716973) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-3.90-purple?style=for-the-badge&logo=academia)](https://doi.org/10.3389/fmolb.2021.716973)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://tulip.rnet.missouri.edu/deepcomplex/web_index.html) 


- **DisVis**: visualizes the accessible interaction space (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/bt) 

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://milou.science.uu.nl/cgi/services/DISVIS/disvis/) 


- **DockStar**: Modelling of multimolecular protein complexes (online)  


    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://bioinfo3d.cs.tau.ac.il/DockStar/) 


- **DockSurf**: A molecular modeling software for the prediction of protein/surface adhesion (eg. gold surfaces) (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.3c00569) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jcim.3c00569)



- **Dockground**: A comprehensive data resource for modeling of protein complexes, datasets, decoys... (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:13-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1002/pro.4481) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.50-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1002/pro.4481)



- **EROS-DOCK**: protein–protein docking using exhaustive branch-and-bound rotational search (standalone)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:9-blue?style=for-the-badge&logo=bookstack)](None) 



- **FRODOCK**: Fast Rotational DOCKing (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:135-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btw141) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/bioinformatics/btw141)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://frodock.chaconlab.org/) 


- **FiberDock, FireDock, FlexDock, PatchDock**: FiberDock: a web server for flexible induced-fit backbone refinement in molecular docking  

    [![Publication](https://img.shields.io/badge/Publication-Citations:86-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/nar/gkq373) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-16.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/nar/gkq373)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://bioinfo3d.cs.tau.ac.il/wk/index.php/Servers_%26_Software) 


- **FiltRest3D**: filter predicted models (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:44-blue?style=for-the-badge&logo=bookstack)](None) 

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://filtrest3d.genesilico.pl/filtrest3d/help.html) 


- **FoldDock**: simultaneous folding and docking protocol FoldDock and AlphaFold (standalone)  
    [![Code](https://img.shields.io/badge/Code-Repository-blue?style=for-the-badge)](https://gitlab.com/ElofssonLab/FoldDock/) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:598-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038/s41467-022-28865-w) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-14.70-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1038/s41467-022-28865-w)



- **GRAMM**: Protein-Protein Docking Web Server (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:35-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1007/978-1-0716-3441-7_5) 

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://gramm.compbio.ku.edu/) 


- **GWIDD**: genome wide docking database (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:14-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1186/1479-7364-6-7) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-3.80-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1186/1479-7364-6-7)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://gwidd.compbio.ku.edu/) 


- **GalaxyDomDock**: An Ab Initio Domain-domain Docking Web Server for Multi-domain Protein Structure Prediction (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:6-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1016/j.jmb.2022.167508) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.70-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1016/j.jmb.2022.167508)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://galaxy.seoklab.org/domdock) 


- **GalaxyHomomer**: protein homo-oligomer structure prediction from a monomer sequence or structure (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:118-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/nar/gkx246) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-16.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/nar/gkx246)



- **GalaxyRefineComplex**: Protein-protein complex structures generated by protein-protein docking methods can be refined (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:109-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038%2Fsrep32153) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-3.80-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1038%2Fsrep32153)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://galaxy.seoklab.org/cgi-bin/submit.cgi?type=COMPLEX) 


- **GalaxyTongDock**: Symmetric and asymmetric ab initio protein-protein docking web server with improved energy parameters (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:707-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/nar/gks493) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-16.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/nar/gks493)



- **GraphPPIS**: Structure-aware protein-protein interaction site prediction using deep graph convolutional network (online), standalone on github  

    [![Publication](https://img.shields.io/badge/Publication-Citations:112-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btab643) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/bioinformatics/btab643)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://biomed.nscc-gz.cn/apps/GraphPPIS) 


- **HDOCK**: hybrid protein–protein and protein–DNA/RNA docking, Pushing the accuracy limit of shape complementarity for protein-protein docking (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:903-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093%2Fnar%2Fgkx407) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-16.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093%2Fnar%2Fgkx407)



- **HMI-PRED**: A Web Server for Structural Prediction of Host-Microbe Interactions Based on Interface Mimicry (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:7-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btac633) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/bioinformatics/btac633)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://interactome.ku.edu.tr/hmi/) 


- **Haddock**: High Ambiguity Driven protein-protein DOCKing (online requires login)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:11-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.21769%2FBioProtoc.3793) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-1.00-purple?style=for-the-badge&logo=academia)](https://doi.org/10.21769%2FBioProtoc.3793)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://haddock.chem.uu.nl/) 


- **HawkDock**: to predict and analyze the protein-protein complex based on computational docking and MM/GBSA (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:442-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/nar/gkz397) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-16.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/nar/gkz397)



- **INTERSPIA**: Explore the Dynamics of Protein-Protein Interactions Among Multiple Species (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:9-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/nar/gky378) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-16.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/nar/gky378)



- **IRaPPA**: pyDockRescoring Server (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:37-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btx068) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/bioinformatics/btx068)



- **InterEvDock2**: protein docking using evolutionary and biological information from homology models and multimeric inputs (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:48-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093%2Fnar%2Fgky377) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-16.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093%2Fnar%2Fgky377)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://bioserv.rpbs.univ-paris-diderot.fr/services/InterEvDock2/) 


- **InterEvDock3**: a combined template-based and free docking server with increased performance through explicit modeling of complex homologs and integration of covariation-based contact maps (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:22-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093%2Fnar%2Fgkab358) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-16.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093%2Fnar%2Fgkab358)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://bioserv.rpbs.univ-paris-diderot.fr/services/InterEvDock3/) 


- **InterPred**: A pipeline to identify and model protein-protein interactions (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:34-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1002/prot.25280) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-3.20-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1002/prot.25280)



- **LightDock**: Artificial Intelligence-powered modeling of macromolecular interactions (macromolecular docking framework, online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:94-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btx555) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/bioinformatics/btx555)



- **P3DOCK**: a protein-RNA docking webserver based on template-based and template-free docking (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:20-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btz478) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/bioinformatics/btz478)



- **PIIMS**: Mutation Hotspot Scanning at the Protein-Protein Interface (online)  




- **PPI-Affinity**: Prediction and Optimization of Protein-Peptide and Protein-Protein Binding Affinity (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:50-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jproteome.2c00020) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-3.80-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jproteome.2c00020)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://protdcal.zmb.uni-due.de/PPIAffinity) 


- **PRODIGY**: PROtein binDIng enerGY prediction (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:913-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btw514) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/bioinformatics/btw514)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://wenmr.science.uu.nl/prodigy/) 


- **PRODIGY-crystal**: classification of biological interfaces in protein complexes (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:32-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btz437) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/bioinformatics/btz437)



- **QASDOM**: Quality ASsessment of DOcking Models, protein-protein (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:10-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btx591) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/bioinformatics/btx591)



- **RosettaDock**: Efficient Flexible Backbone Protein-Protein Docking for Challenging Targets (standalone and online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:503-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093%2Fnar%2Fgkn216) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-16.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093%2Fnar%2Fgkn216)



- **SPServer**: split-statistical potentials for the analysis of protein structures and protein-protein interactions (PPI) (online) (or http://sbi.upf.edu/spserver/)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:9-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1186/s12859-020-03770-5) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-2.90-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1186/s12859-020-03770-5)



- **SwarmDock**: webservice for generating 3D structures of protein-protein complexes (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:1-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1007/978-1-0716-0708-4_11) 

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://bmm.crick.ac.uk/~SwarmDock/) 


- **ZDOCK**: docking (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:528-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1371/journal.pone.0024657) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-2.90-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1371/journal.pone.0024657)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://zdock.umassmed.edu/) 


- **ezCADD**: a Rapid 2D/3D Visualization Enabled Web Modeling Environment for Democratizing Computer-Aided Drug Design (small molecule docking, protein docking, 2D/3D interaction viz, de novo lead optimization, pocket search and polypharmacology) online  

    [![Publication](https://img.shields.io/badge/Publication-Citations:61-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.8b00633) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jcim.8b00633)



- **pyDock**: pyDockWEB: a web server for rigid-body protein–protein docking using electrostatics and desolvation scoring  

    [![Publication](https://img.shields.io/badge/Publication-Citations:241-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btt262) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/bioinformatics/btt262)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://life.bsc.es/pid/pydock/) 


- **pyDockDNA**: A new web server for energy-based protein-DNA docking and scoring (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:21-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.3389/fmolb.2022.988996) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-3.90-purple?style=for-the-badge&logo=academia)](https://doi.org/10.3389/fmolb.2022.988996)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://model3dbio.csic.es/pydockdna) 


- **pyDockEneRes**: per-residue decomposition of protein-protein docking energy (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:20-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btz884) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/bioinformatics/btz884)



- **webSDA**: Simulation of Diffusional Assosiation (SDA) (online)  

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/nar/gk) 

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://websda.h-its.org/webSDA) 

## **Quality Assessment**


- **DockQ**: A Quality Measure for Protein-Protein Docking Models (standalone)  
    [![Code](https://img.shields.io/github/stars/bjornwallner/DockQ/?style=for-the-badge&logo=github)](https://github.com/bjornwallner/DockQ/) 
    [![Last Commit](https://img.shields.io/github/last-commit/bjornwallner/DockQ/?style=for-the-badge&logo=github)](https://github.com/bjornwallner/DockQ/) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:306-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1371/journal.pone.0161879) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-2.90-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1371/journal.pone.0161879)


