## General

- **[AlphaFold](https://alphafold.ebi.ac.uk/)**: Protein Structure Database (predicted human 3D proteins, July 2021)
	- **[AlphaFold-Multimer](https://colab.research.google.com/github/deepmind/alphafold/blob/main/notebooks/AlphaFold.ipynb)**: a model that can predict the structure of multi-chain protein complexes with increased accuracy (standalone)
	- **[Openfold](https://github.com/aqlaboratory/openfold)**: A faithful PyTorch reproduction of DeepMind's AlphaFold2 (standalone)

- **[ESMFold](https://esmatlas.com/about)**: ESM Metagenomic Atlas contains several millions of predicted protein structures (can be used via ChimeraX) (online)
- **[ESM](https://github.com/facebookresearch/esm)**: Evolutionary Scale Modeling to predict protein 3D structure (standalone) (online)
- **[ColabFold](https://github.com/sokrypton/ColabFold)**: making protein folding accessible to all (standalone)

- **[RoseTTAFold](https://github.com/RosettaCommons/RoseTTAFold)**: This package contains deep learning models and scripts for RoseTTAFold, an accurate method for protein structure prediction that includes a 3-track network. It's capable of modeling multi-chain complexes and provides an option for large scale sampling.
- **[McGuffin Group Web Servers](https://www.reading.ac.uk/bioinf/index.html)**: This link points to the home page of the McGuffin Group Web Servers at the University of Reading, which provides various bioinformatics tools, although specific details about the tools were not provided.
- **[Robetta](https://robetta.bakerlab.org/)**: Robetta offers structure prediction using deep learning methods like RoseTTAFold and TrRosetta. It allows for custom sequence alignments, constraints, local fragments for homology modeling, and can model multi-chain complexes.
- **[DMFold](https://zhanggroup.org/DMFold/download/)**: DMFold standalone package is an integrated program of DeepMSA2 and AlphaFold2 for protein monomer and protein complex structure prediction
- **[Raptor-X](https://github.com/AndersJing/RaptorX-Single)**: RaptorX-Single: exploring the advantage of single sequence based protein structure prediction
- **[Evo](https://github.com/evo-design/evo)**: A long-context foundation model that generalizes across the central dogma of biology: DNA, RNA, and proteins.
- **[DMFold](https://zhanggroup.org/DMFold/download/)**: A tool that integrates large genomic and metagenomics sequence databases for improved protein structure prediction.
- **[]()**


## **Trans-membrane Proteins**

- **[PredMP](http://www.predmp.com/)**: The website's specific content was not retrievable due to request issues, but PredMP is designed for predicting membrane protein types and orientations.
- **[MemBrain](http://www.csbio.sjtu.edu.cn/bioinf/MemBrain/)**: The site provides resources related to membrane protein prediction but specific details about the MemBrain tool were not provided.
- **[RosettaGPCR](https://github.com/benderb1/rosettagpcr)**: This repository contains methods for generating models of G protein-coupled receptors (GPCRs) using Rosetta, including a database of templates updated through June 2020.
- **[membraneFold](https://ku.biolib.com/MembraneFold/)**: This resource is intended for predicting membrane protein structures, but specific details were not provided.

## **Quaternary structure**

- **[AlphaFold-multimer](https://github.com/deepmind/alphafold)**: The specific details about AlphaFold-multimer could not be directly retrieved, but it extends AlphaFold2's capabilities to predict structures of protein complexes (multimers).
- **[DeepComplex](http://tulip.rnet.missouri.edu/deepcomplex/web_index.html)**: This document has moved, and direct content was not provided in the data fetched, but DeepComplex is aimed at predicting quaternary protein structures.
- **[CombFold](https://lnkd.in/gRVdfaZV)**: a combinatorial and hierarchical assembly algorithm combined with AlphaFold2 for predicting structures of large protein assemblies

## Structure Prediction with Ligand

- **[Umol](https://github.com/patrickbryant1/Umol)**: Umol is designed for protein-ligand structure prediction, representing the protein with a multiple sequence alignment and the ligand as a SMILES string, with versions utilizing protein pocket information recommended.
- **[RF_Diffusion_All_Atom](https://github.com/baker-laboratory/rf_diffusion_all_atom)**: A generative model for protein design
- **[NeuralPlexer](https://github.com/zrqiao/NeuralPLexer)**: a deep generative model to jointly predict protein-ligand complex 3D structures and beyond.
- **[RoseTTAFold_AllAtom](https://github.com/AaronFeller/RoseTTAFold-All-Atom/blob/main/README.md)**: biomolecular structure prediction neural network that can predict a broad range of biomolecular assemblies including proteins, nucleic acids, small molecules, covalent modifications and metals as outlined in the RFAA paper.
- **[DynamicBind](https://github.com/luwei0917/DynamicBind)**: DynamicBind recovers ligand-specific conformations from unbound protein structures (e.g. AF2-predicted structures), promoting efficient transitions between different equilibrium states.
- **[DiffusionProteinLigand](https://github.com/shuyana/DiffusionProteinLigand)**: End-to-end proteinâ€“ligand complex structure generation with diffusion-based generative models (standalone).