---
icon: simple/abbrobotstudio
---




- **OpenVS**: An artificial intelligence accelerated virtual screening platform for drug discovery  
    [![Code](https://img.shields.io/github/stars/gfzhou/OpenVS?style=for-the-badge&logo=github)](https://github.com/gfzhou/OpenVS) 
    [![Last Commit](https://img.shields.io/github/last-commit/gfzhou/OpenVS?style=for-the-badge&logo=github)](https://github.com/gfzhou/OpenVS) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:23-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038/s41467-024-52061-7) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-14.70-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1038/s41467-024-52061-7)


## **All-Atom-DL methods**


- **AlphaFold3**: Accurate structure prediction of biomolecular interactions with AlphaFold 3  

    [![Publication](https://img.shields.io/badge/Publication-Citations:2675-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038/s41586-024-07487-w) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-50.50-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1038/s41586-024-07487-w)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://golgi.sandbox.google.com/) 


- **Boltz-1**: Boltz-1 is an open-source model which predicts the 3D structure of proteins, RNA, DNA and small molecules  
    [![Code](https://img.shields.io/github/stars/jwohlwend/boltz?tab=readme-ov-file?style=for-the-badge&logo=github)](https://github.com/jwohlwend/boltz?tab=readme-ov-file) 
    [![Last Commit](https://img.shields.io/github/last-commit/jwohlwend/boltz?tab=readme-ov-file?style=for-the-badge&logo=github)](https://github.com/jwohlwend/boltz?tab=readme-ov-file) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:20-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1101/2024.11.19.624167) 



- **DiffusionProteinLigand**: End-to-end protein–ligand complex structure generation with diffusion-based generative models (standalone).  
    [![Code](https://img.shields.io/github/stars/shuyana/DiffusionProteinLigand?style=for-the-badge&logo=github)](https://github.com/shuyana/DiffusionProteinLigand) 
    [![Last Commit](https://img.shields.io/github/last-commit/shuyana/DiffusionProteinLigand?style=for-the-badge&logo=github)](https://github.com/shuyana/DiffusionProteinLigand) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:22-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1186/s12859-023-05354-5) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-2.90-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1186/s12859-023-05354-5)



- **DockFormer**: employs fully flexible docking, where no part of the receptor remains rigid, by adapting the AlphaFold2 architecture  
    [![Code](https://img.shields.io/badge/Code-Repository-blue?style=for-the-badge)](https://huggingface.co/spaces/benshor/DockFormer) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1101/2024.11.25.625135) 



- **DynamicBind**: DynamicBind recovers ligand-specific conformations from unbound protein structures (e.g. AF2-predicted structures), promoting efficient transitions between different equilibrium states.  
    [![Code](https://img.shields.io/github/stars/luwei0917/DynamicBind?style=for-the-badge&logo=github)](https://github.com/luwei0917/DynamicBind) 
    [![Last Commit](https://img.shields.io/github/last-commit/luwei0917/DynamicBind?style=for-the-badge&logo=github)](https://github.com/luwei0917/DynamicBind) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:48-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038/s41467-024-45461-2) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-14.70-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1038/s41467-024-45461-2)



- **FlowDock**: Geometric Flow Matching for Generative Protein-Ligand Docking and Affinity Prediction  
    [![Code](https://img.shields.io/github/stars/BioinfoMachineLearning/FlowDock?style=for-the-badge&logo=github)](https://github.com/BioinfoMachineLearning/FlowDock) 
    [![Last Commit](https://img.shields.io/github/last-commit/BioinfoMachineLearning/FlowDock?style=for-the-badge&logo=github)](https://github.com/BioinfoMachineLearning/FlowDock) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.48550/arXiv.2412.10966) 



- **NeuralPlexer**: deep generative model to jointly predict protein-ligand complex 3D structures and beyond.  
    [![Code](https://img.shields.io/github/stars/zrqiao/NeuralPLexer?style=for-the-badge&logo=github)](https://github.com/zrqiao/NeuralPLexer) 
    [![Last Commit](https://img.shields.io/github/last-commit/zrqiao/NeuralPLexer?style=for-the-badge&logo=github)](https://github.com/zrqiao/NeuralPLexer) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:47-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038/s42256-024-00792-z) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-18.80-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1038/s42256-024-00792-z)



- **RoseTTAFold-AllAtom**: RoseTTAFold All-Atom (RFAA) is a deep network capable of modeling full biological assemblies containing proteins, nucleic acids, small molecules, metals, and covalent modifications with high accuracy.  

    [![Publication](https://img.shields.io/badge/Publication-Citations:269-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1126/science.adl2528) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-44.70-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1126/science.adl2528)


### **Reviews**

- **Have protein-ligand co-folding methods moved beyond memorisation?**:   

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1101/2025.02.03.636309) 


## **Allosteric sites**


- **FASTDock**: FASTDock is a pipeline for allosteric drug discovery, offering scripts and a Jupyter notebook for efficiently generating and analyzing docking grids, clusters, and fingerprint screenings.  
    [![Code](https://img.shields.io/github/stars/BrooksResearchGroup-UM/FASTDock?style=for-the-badge&logo=github)](https://github.com/BrooksResearchGroup-UM/FASTDock) 
    [![Last Commit](https://img.shields.io/github/last-commit/BrooksResearchGroup-UM/FASTDock?style=for-the-badge&logo=github)](https://github.com/BrooksResearchGroup-UM/FASTDock) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.3c00895.s001) 


## **Blind Docking**


- **BINDSURF**: High-Throughput parallel blind Virtual Screening  

    [![Publication](https://img.shields.io/badge/Publication-Citations:144-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1186/1471-2105-13-S14-S13) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-2.90-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1186/1471-2105-13-S14-S13)



- **CBDock2**: CBDock2 is an improved protein-ligand blind docking tool integrating cavity detection, docking, and homologous template fitting to suggest novel therapeutic targets for biological and pharmaceutical studies.  

    [![Publication](https://img.shields.io/badge/Publication-Citations:589-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/nar/gkac394) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-16.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/nar/gkac394)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://cadd.labshare.cn/cb-dock2/php/index.php) 


- **CoBDock**: CoBDock is a reference implementation of the COBDock algorithm, detailing steps for setup and execution on Linux, with a focus on integrating various molecular docking and pocket identification algorithms.  
    [![Code](https://img.shields.io/github/stars/DavidMcDonald1993/cobdock?style=for-the-badge&logo=github)](https://github.com/DavidMcDonald1993/cobdock) 
    [![Last Commit](https://img.shields.io/github/last-commit/DavidMcDonald1993/cobdock?style=for-the-badge&logo=github)](https://github.com/DavidMcDonald1993/cobdock) 




- **METADOCK2**: high-throughput parallel metaheuristic scheme for molecular docking  
    [![Code](https://img.shields.io/badge/Code-Repository-blue?style=for-the-badge)](https://Baldoimbernon@bitbucket.org/Baldoimbernon/metadock_2.git) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:12-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btz958) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/bioinformatics/btz958)


## **Chemical Space Docking**


- **SpaceHASTEN**: SpaceHASTEN: A structure-based virtual screening tool for non-enumerated virtual chemical libraries  
    [![Code](https://img.shields.io/github/stars/http://github.com/TuomoKalliokoski/SpaceHASTEN?style=for-the-badge&logo=github)](http://github.com/TuomoKalliokoski/SpaceHASTEN) 
    [![Last Commit](https://img.shields.io/github/last-commit/http://github.com/TuomoKalliokoski/SpaceHASTEN?style=for-the-badge&logo=github)](http://github.com/TuomoKalliokoski/SpaceHASTEN) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.4c01790) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jcim.4c01790)


## **Classical**


- **AutoDock Vina**: AutoDock Vina 1.2.0: New Docking Methods, Expanded Force Field, and Python Bindings  
    [![Code](https://img.shields.io/github/stars/ccsb-scripps/AutoDock-Vina?style=for-the-badge&logo=github)](https://github.com/ccsb-scripps/AutoDock-Vina) 
    [![Last Commit](https://img.shields.io/github/last-commit/ccsb-scripps/AutoDock-Vina?style=for-the-badge&logo=github)](https://github.com/ccsb-scripps/AutoDock-Vina) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:3123-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.1c00203) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jcim.1c00203)



- **BR-NiB**: Brute Force Negative Image-Based Optimization. A docking rescoring method that relyes on shape/ electrostatic potential similarity between the docking poses of ligands and the cavity-based negative images (standalone).  
    [![Code](https://img.shields.io/github/stars/jvlehtonen/brutenib?style=for-the-badge&logo=github)](https://github.com/jvlehtonen/brutenib) 
    [![Last Commit](https://img.shields.io/github/last-commit/jvlehtonen/brutenib?style=for-the-badge&logo=github)](https://github.com/jvlehtonen/brutenib) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:3-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.1c01145) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jcim.1c01145)



- **DSDP**: DSDP: A Blind Docking Strategy Accelerated by GPUs  
    [![Code](https://img.shields.io/github/stars/PKUGaoGroup/DSDP?style=for-the-badge&logo=github)](https://github.com/PKUGaoGroup/DSDP) 
    [![Last Commit](https://img.shields.io/github/last-commit/PKUGaoGroup/DSDP?style=for-the-badge&logo=github)](https://github.com/PKUGaoGroup/DSDP) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:13-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.3c00519) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jcim.3c00519)



- **DockThor**:   

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](None) 



- **FWAVina**: The content for FWAVina was not accessible due to restrictions or an error from the URL provided.  
    [![Code](https://img.shields.io/github/stars/eddyblue/FWAVina/tree/master?style=for-the-badge&logo=github)](https://github.com/eddyblue/FWAVina/tree/master) 
    [![Last Commit](https://img.shields.io/github/last-commit/eddyblue/FWAVina/tree/master?style=for-the-badge&logo=github)](https://github.com/eddyblue/FWAVina/tree/master) 




- **GLOW-IVES**: Provides Python implementation of GLOW (auGmented sampLing with softened vdW potential) and IVES (Iterative Ensemble Sampling) protocols for pose sampling, along with new cross-docking datasets.  
    [![Code](https://img.shields.io/github/stars/drorlab/GLOW_IVES?style=for-the-badge&logo=github)](https://github.com/drorlab/GLOW_IVES) 
    [![Last Commit](https://img.shields.io/github/last-commit/drorlab/GLOW_IVES?style=for-the-badge&logo=github)](https://github.com/drorlab/GLOW_IVES) 




- **GeauxDock**: An ultra-fast automated docking program from LSU, predicting how small ligands bind to macromolecules using a novel hybrid force field and a Monte Carlo protocol.  
    [![Code](https://img.shields.io/badge/Code-Repository-blue?style=for-the-badge)](http://www.brylinski.org/geauxdock) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:24-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1371/journal.pone.0158898) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-2.90-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1371/journal.pone.0158898)



- **HESS**: A new protein-ligand docking software with an improved method of molecular conformation optimization  
    [![Code](https://img.shields.io/github/stars/Entroforce/Hess?style=for-the-badge&logo=github)](https://github.com/Entroforce/Hess) 
    [![Last Commit](https://img.shields.io/github/last-commit/Entroforce/Hess?style=for-the-badge&logo=github)](https://github.com/Entroforce/Hess) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.26434/chemrxiv-2023-jh1kk) 



- **JAMDA**: Redocking the PDB  

    [![Publication](https://img.shields.io/badge/Publication-Citations:12-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.3c01573) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jcim.3c01573)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://proteins.plus/) 


- **MzDock**: MzDOCK: A free ready‐to‐use GUI‐based pipeline for molecular docking simulations  
    [![Code](https://img.shields.io/github/stars/Muzatheking12/MzDOCK?style=for-the-badge&logo=github)](https://github.com/Muzatheking12/MzDOCK) 
    [![Last Commit](https://img.shields.io/github/last-commit/Muzatheking12/MzDOCK?style=for-the-badge&logo=github)](https://github.com/Muzatheking12/MzDOCK) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:5-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1002/jcc.27390) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-3.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1002/jcc.27390)



- **Opendock**: OpenDock: A pytorch-based open-source framework for protein-ligand docking and modelling  
    [![Code](https://img.shields.io/github/stars/guyuehuo/opendock?style=for-the-badge&logo=github)](https://github.com/guyuehuo/opendock) 
    [![Last Commit](https://img.shields.io/github/last-commit/guyuehuo/opendock?style=for-the-badge&logo=github)](https://github.com/guyuehuo/opendock) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btae628) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/bioinformatics/btae628)



- **Paradocks**: parallel docking suite  
    [![Code](https://img.shields.io/github/stars/cbaldauf/paradocks?style=for-the-badge&logo=github)](https://github.com/cbaldauf/paradocks) 
    [![Last Commit](https://img.shields.io/github/last-commit/cbaldauf/paradocks?style=for-the-badge&logo=github)](https://github.com/cbaldauf/paradocks) 




- **ProBiS-Dock**: A Hybrid Multitemplate Homology Flexible Docking Algorithm Enabled by Protein Binding Site Comparison (standalone).  
    [![Code](https://img.shields.io/badge/Code-Repository-blue?style=for-the-badge)](https://gitlab.com/janezkonc/insidrug/-/releases/probis_dock_algorithm_v1.0.0) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:5-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.1c01176) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jcim.1c01176)



- **QVINA**: QuickVina 2 aims to accurately speed up AutoDock Vina, providing up to 20.49-fold acceleration with high correlation in binding energy prediction.  
    [![Code](https://img.shields.io/github/stars/QVina/qvina?style=for-the-badge&logo=github)](https://github.com/QVina/qvina) 
    [![Last Commit](https://img.shields.io/github/last-commit/QVina/qvina?style=for-the-badge&logo=github)](https://github.com/QVina/qvina) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/bt) 



- **RDPSO_Vina**: A fast docking tool utilizing random drift particle swarm optimization based on the AutoDock Vina and PSOVina framework.  
    [![Code](https://img.shields.io/github/stars/li-jin-xing/RDPSOVina?style=for-the-badge&logo=github)](https://github.com/li-jin-xing/RDPSOVina) 
    [![Last Commit](https://img.shields.io/github/last-commit/li-jin-xing/RDPSOVina?style=for-the-badge&logo=github)](https://github.com/li-jin-xing/RDPSOVina) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:3-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1007/s10822-022-00455-4) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-3.00-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1007/s10822-022-00455-4)



- **RxDock**: a fast, versatile, and open-source program for docking ligands to proteins and nucleic acids (standalone).  
    [![Code](https://img.shields.io/github/stars/rxdock/rxdock?style=for-the-badge&logo=github)](https://github.com/rxdock/rxdock) 
    [![Last Commit](https://img.shields.io/github/last-commit/rxdock/rxdock?style=for-the-badge&logo=github)](https://github.com/rxdock/rxdock) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:457-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1371/journal.pcbi.1003571) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-3.80-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1371/journal.pcbi.1003571)



- **SMINA**: A fork of AutoDock Vina that supports scoring function development and high-performance energy minimization, maintained by the University of Pittsburgh.  
    [![Code](https://img.shields.io/badge/Code-Repository-blue?style=for-the-badge)](https://sourceforge.net/projects/smina/) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:791-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/ci300604z) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/ci300604z)



- **Surflex-Tools**: starting with version 4 (standalone).  




- **Uni-Dock**: Uni-Dock is a GPU-accelerated molecular docking program that supports various scoring functions and achieves significant speed-up compared with AutoDock Vina on a single CPU core.  
    [![Code](https://img.shields.io/github/stars/dptech-corp/Uni-Dock?style=for-the-badge&logo=github)](https://github.com/dptech-corp/Uni-Dock) 
    [![Last Commit](https://img.shields.io/github/last-commit/dptech-corp/Uni-Dock?style=for-the-badge&logo=github)](https://github.com/dptech-corp/Uni-Dock) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:35-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jctc.2c01145) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.70-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jctc.2c01145)



- **VinaCarb**: The content for VinaCarb was not available from the URL provided.  

    [![Publication](https://img.shields.io/badge/Publication-Citations:102-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jctc.5b00834) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.70-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jctc.5b00834)



- **VinaGPU2.0**: Vina-GPU 2.0: Further Accelerating AutoDock Vina and Its Derivatives with Graphics Processing Units  
    [![Code](https://img.shields.io/github/stars/DeltaGroupNJUPT/Vina-GPU-2.0?style=for-the-badge&logo=github)](https://github.com/DeltaGroupNJUPT/Vina-GPU-2.0) 
    [![Last Commit](https://img.shields.io/github/last-commit/DeltaGroupNJUPT/Vina-GPU-2.0?style=for-the-badge&logo=github)](https://github.com/DeltaGroupNJUPT/Vina-GPU-2.0) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:50-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.2c01504) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jcim.2c01504)



- **VinaXB (halogen-bonding)**: Introduces a halogen bonding scoring function (XBSF) in AutoDock Vina, termed AutoDock VinaXB, to improve docking accuracy with halogenated ligands.  
    [![Code](https://img.shields.io/github/stars/sirimullalab/vinaXB/tree/master?style=for-the-badge&logo=github)](https://github.com/sirimullalab/vinaXB/tree/master) 
    [![Last Commit](https://img.shields.io/github/last-commit/sirimullalab/vinaXB/tree/master?style=for-the-badge&logo=github)](https://github.com/sirimullalab/vinaXB/tree/master) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:107-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1186/s13321-016-0139-1) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-7.10-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1186/s13321-016-0139-1)



- **Xdock**: versatile docking framework built for both protein–ligand and nucleic acid–ligand interactions  
    [![Code](https://img.shields.io/badge/Code-Repository-blue?style=for-the-badge)](http://huanglab.phys.hust.edu.cn/XDock/download/) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.4c00855) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jcim.4c00855)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://huanglab.phys.hust.edu.cn/XDock/) 


- **labodock**: LABODOCK offers a collection of Jupyter Notebooks for molecular docking on Google Colab with minimal coding, streamlining pre- and post-docking processes.  
    [![Code](https://img.shields.io/github/stars/RyanZR/labodock?style=for-the-badge&logo=github)](https://github.com/RyanZR/labodock) 
    [![Last Commit](https://img.shields.io/github/last-commit/RyanZR/labodock?style=for-the-badge&logo=github)](https://github.com/RyanZR/labodock) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.5281/zenodo.8246977-blue) 



- **parallel-PLANTS**: Offers a method for parallel molecular docking using the PLANTS software, aimed at academic use.  
    [![Code](https://img.shields.io/github/stars/discoverdata/parallel-PLANTS?style=for-the-badge&logo=github)](https://github.com/discoverdata/parallel-PLANTS) 
    [![Last Commit](https://img.shields.io/github/last-commit/discoverdata/parallel-PLANTS?style=for-the-badge&logo=github)](https://github.com/discoverdata/parallel-PLANTS) 




- **restretto**: Effective Protein–Ligand Docking Strategy via Fragment Reuse and a Proof-of-Concept Implementation (standalone).  
    [![Code](https://img.shields.io/github/stars/akiyamalab/restretto?style=for-the-badge&logo=github)](https://github.com/akiyamalab/restretto) 
    [![Last Commit](https://img.shields.io/github/last-commit/akiyamalab/restretto?style=for-the-badge&logo=github)](https://github.com/akiyamalab/restretto) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:3-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acsomega.2c03470) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-3.70-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acsomega.2c03470)



- **vina4dv**:   
    [![Code](https://img.shields.io/github/stars/chengwang88/vina4dv?style=for-the-badge&logo=github)](https://github.com/chengwang88/vina4dv) 
    [![Last Commit](https://img.shields.io/github/last-commit/chengwang88/vina4dv?style=for-the-badge&logo=github)](https://github.com/chengwang88/vina4dv) 



## **Consensus**


- **DockingPie**: DockingPie is a PyMOL plugin that facilitates consensus docking and scoring analyses, integrating four docking programs (Smina, Autodock Vina, RxDock, and ADFR) to offer a versatile platform for molecular and consensus docking.  
    [![Code](https://img.shields.io/github/stars/paiardin/DockingPie?style=for-the-badge&logo=github)](https://github.com/paiardin/DockingPie) 
    [![Last Commit](https://img.shields.io/github/last-commit/paiardin/DockingPie?style=for-the-badge&logo=github)](https://github.com/paiardin/DockingPie) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:35-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btac452) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/bioinformatics/btac452)



- **Exponential Consensus Ranking**:   

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](None) 



- **VoteDock**: VoteDock: Consensus docking method for prediction of protein–ligand interactions  

    [![Publication](https://img.shields.io/badge/Publication-Citations:75-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1002/jcc.21642) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-3.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1002/jcc.21642)



- **dockECR**: dockECR: Open consensus docking and ranking protocol for virtual screening of small molecules  
    [![Code](https://img.shields.io/github/stars/rochoa85/dockECR?style=for-the-badge&logo=github)](https://github.com/rochoa85/dockECR) 
    [![Last Commit](https://img.shields.io/github/last-commit/rochoa85/dockECR?style=for-the-badge&logo=github)](https://github.com/rochoa85/dockECR) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:15-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1016/j.jmgm.2021.108023) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-2.70-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1016/j.jmgm.2021.108023)


### **Reviews**

- **Multi-Criteria Decision Analysis in Drug Discovery**:   

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.3390/applbiosci4010002) 


## **Electron Density**


- **CryoXKit**: Docking guidance with experimental ligand structural density  
    [![Code](https://img.shields.io/github/stars/forlilab/CryoXKit?style=for-the-badge&logo=github)](https://github.com/forlilab/CryoXKit) 
    [![Last Commit](https://img.shields.io/github/last-commit/forlilab/CryoXKit?style=for-the-badge&logo=github)](https://github.com/forlilab/CryoXKit) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1002/pro.70082) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.50-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1002/pro.70082)


## **Flexible Docking**


- **ADFR**: AutoDockFR is a protein-ligand docking program supporting selective receptor flexibility and covalent docking, part of the ADFR suite for streamlined docking procedures.  

    [![Publication](https://img.shields.io/badge/Publication-Citations:415-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1371/journal.pcbi.1004586) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-3.80-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1371/journal.pcbi.1004586)



- **ApoDock**:   
    [![Code](https://img.shields.io/github/stars/ld139/ApoDock_public?style=for-the-badge&logo=github)](https://github.com/ld139/ApoDock_public) 
    [![Last Commit](https://img.shields.io/github/last-commit/ld139/ApoDock_public?style=for-the-badge&logo=github)](https://github.com/ld139/ApoDock_public) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jctc.4c01636) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.70-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jctc.4c01636)



- **DSDPFlex**: DSDPFlex: An Improved Flexible-Receptor Docking Method with GPU Acceleration  
    [![Code](https://img.shields.io/github/stars/PKUGaoGroup/DSDP?style=for-the-badge&logo=github)](https://github.com/PKUGaoGroup/DSDP) 
    [![Last Commit](https://img.shields.io/github/last-commit/PKUGaoGroup/DSDP?style=for-the-badge&logo=github)](https://github.com/PKUGaoGroup/DSDP) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.4c01715) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jcim.4c01715)



- **FABFlex**: Fast and Accurate Blind Flexible Docking  
    [![Code](https://img.shields.io/github/stars/tmlr-group/FABFlex?style=for-the-badge&logo=github)](https://github.com/tmlr-group/FABFlex) 
    [![Last Commit](https://img.shields.io/github/last-commit/tmlr-group/FABFlex?style=for-the-badge&logo=github)](https://github.com/tmlr-group/FABFlex) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.48550/arXiv.2502.14934) 



- **FlexAID**: FlexAID: Revisiting Docking on Non-Native-Complex Structures  
    [![Code](https://img.shields.io/github/stars/NRGlab/FlexAID?style=for-the-badge&logo=github)](https://github.com/NRGlab/FlexAID) 
    [![Last Commit](https://img.shields.io/github/last-commit/NRGlab/FlexAID?style=for-the-badge&logo=github)](https://github.com/NRGlab/FlexAID) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:55-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.5b00078) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jcim.5b00078)



- **GNINA**: GNINA is a molecular docking program that incorporates scoring and optimization of ligands using convolutional neural networks, aiming to combine the versatility of smina and AutoDock Vina with the predictive power of deep learning.  
    [![Code](https://img.shields.io/github/stars/gnina/gnina?style=for-the-badge&logo=github)](https://github.com/gnina/gnina) 
    [![Last Commit](https://img.shields.io/github/last-commit/gnina/gnina?style=for-the-badge&logo=github)](https://github.com/gnina/gnina) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:333-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1186/s13321-021-00522-2) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-7.10-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1186/s13321-021-00522-2)



- **PackDock**: Describes PackDock as a diffusion-based side chain packing model for flexible protein-ligand docking, indicating code will be available following the publication of their paper.  
    [![Code](https://img.shields.io/github/stars/Zhang-Runze/PackDock?style=for-the-badge&logo=github)](https://github.com/Zhang-Runze/PackDock) 
    [![Last Commit](https://img.shields.io/github/last-commit/Zhang-Runze/PackDock?style=for-the-badge&logo=github)](https://github.com/Zhang-Runze/PackDock) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.21203/rs.3.rs-4128729) 



- **hybrid-SA-IFD**: Robust Induced Fit Docking Approach with the Combination ofthe Hybrid All-Atom/United-Atom/Coarse-Grained Model andSimulated Annealing  
    [![Code](https://img.shields.io/github/stars/ldx022/hybrid-SA-IFD?style=for-the-badge&logo=github)](https://github.com/ldx022/hybrid-SA-IFD) 
    [![Last Commit](https://img.shields.io/github/last-commit/ldx022/hybrid-SA-IFD?style=for-the-badge&logo=github)](https://github.com/ldx022/hybrid-SA-IFD) 




- **iDock**: iDock is a multithreaded virtual screening tool for flexible ligand docking in computational drug discovery, inspired by AutoDock Vina and hosted on GitHub under Apache License 2.0.  
    [![Code](https://img.shields.io/github/stars/gloglita/idock?style=for-the-badge&logo=github)](https://github.com/gloglita/idock) 
    [![Last Commit](https://img.shields.io/github/last-commit/gloglita/idock?style=for-the-badge&logo=github)](https://github.com/gloglita/idock) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:71-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1109/CIBCB.2012.6217214) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1109/CIBCB.2012.6217214)



- **tiny_IFD**: Offers lightweight induced fit docking capabilities.  
    [![Code](https://img.shields.io/github/stars/darrenjhsu/tiny_IFD?style=for-the-badge&logo=github)](https://github.com/darrenjhsu/tiny_IFD) 
    [![Last Commit](https://img.shields.io/github/last-commit/darrenjhsu/tiny_IFD?style=for-the-badge&logo=github)](https://github.com/darrenjhsu/tiny_IFD) 



## **Fragment-based**


- **SEED**: fragment docking with force-field based evaluation of binding energy  
    [![Code](https://img.shields.io/badge/Code-Repository-blue?style=for-the-badge)](https://gitlab.com/CaflischLab/SEED) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:133-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1002/(SICI)1097-0134(19991001)37:1%3C88::AID-PROT9%3E3.0.CO;2-O) 



- **Spresso**: an ultrafast compound pre-screening method based on compound decomposition  

    [![Publication](https://img.shields.io/badge/Publication-Citations:13-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btx178) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/bioinformatics/btx178)


## **HPC enabled**


- **EasyDock**: EasyDock is a fully automatic pipeline for molecular docking, featuring a simple setup for using vina and gnina/smina for docking operations, intended to streamline the docking process.  
    [![Code](https://img.shields.io/github/stars/ci-lab-cz/easydock/tree/master?style=for-the-badge&logo=github)](https://github.com/ci-lab-cz/easydock/tree/master) 
    [![Last Commit](https://img.shields.io/github/last-commit/ci-lab-cz/easydock/tree/master?style=for-the-badge&logo=github)](https://github.com/ci-lab-cz/easydock/tree/master) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:10-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1186/s13321-023-00772-2) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-7.10-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1186/s13321-023-00772-2)



- **POAP**: POAP: A GNU parallel based multithreaded pipeline of open babel and AutoDock suite for boosted high throughput virtual screening  
    [![Code](https://img.shields.io/github/stars/inpacdb/POAP?style=for-the-badge&logo=github)](https://github.com/inpacdb/POAP) 
    [![Last Commit](https://img.shields.io/github/last-commit/inpacdb/POAP?style=for-the-badge&logo=github)](https://github.com/inpacdb/POAP) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:67-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1016/j.compbiolchem.2018.02.012) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-2.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1016/j.compbiolchem.2018.02.012)



- **VinaLC**:   
    [![Code](https://img.shields.io/github/stars/XiaohuaZhangLLNL/VinaLC?style=for-the-badge&logo=github)](https://github.com/XiaohuaZhangLLNL/VinaLC) 
    [![Last Commit](https://img.shields.io/github/last-commit/XiaohuaZhangLLNL/VinaLC?style=for-the-badge&logo=github)](https://github.com/XiaohuaZhangLLNL/VinaLC) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](None) 



- **VinaMPI**: VinaMPI: Facilitating multiple receptor high-throughput virtual docking on high-performance computers  

    [![Publication](https://img.shields.io/badge/Publication-Citations:69-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1002/jcc.23367) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-3.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1002/jcc.23367)



- **VinaSC**: VinaSC: Scalable Autodock Vina with fine-grained scheduling on heterogeneous platform  

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1109/BIBM.2016.7822624) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1109/BIBM.2016.7822624)


## **MD-based**


- **ColDock**: ColDock: Concentrated Ligand Docking with All-Atom Molecular Dynamics Simulation  

    [![Publication](https://img.shields.io/badge/Publication-Citations:15-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jpcb.8b02756) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-2.80-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jpcb.8b02756)


## **ML-based**


- **3DMolFormer**: A Dual-channel Framework for Structure-based Drug Discovery  
    [![Code](https://img.shields.io/github/stars/HXYfighter/3DMolFormer?style=for-the-badge&logo=github)](https://github.com/HXYfighter/3DMolFormer) 
    [![Last Commit](https://img.shields.io/github/last-commit/HXYfighter/3DMolFormer?style=for-the-badge&logo=github)](https://github.com/HXYfighter/3DMolFormer) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.48550/arXiv.2502.05107) 



- **AQDNet**: Implements a Deep Neural Network for Protein-Ligand Docking Simulation, focusing on identifying correct binding poses through convolutional neural network approaches.  
    [![Code](https://img.shields.io/github/stars/koji11235/AQDnet?style=for-the-badge&logo=github)](https://github.com/koji11235/AQDnet) 
    [![Last Commit](https://img.shields.io/github/last-commit/koji11235/AQDnet?style=for-the-badge&logo=github)](https://github.com/koji11235/AQDnet) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:5-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acsomega.3c02411) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-3.70-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acsomega.3c02411)



- **ArtiDock**: ArtiDock: fast and accurate machine learning approach to protein-ligand docking based on multimodal data augmentation  

    [![Publication](https://img.shields.io/badge/Publication-Citations:1-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1101/2024.03.14.585019) 



- **CarsiDock**: a deep learning paradigm for accurate protein–ligand docking and screening based on large-scale pre-training (standalone, 2024).  
    [![Code](https://img.shields.io/github/stars/carbonsilicon-ai/CarsiDock?style=for-the-badge&logo=github)](https://github.com/carbonsilicon-ai/CarsiDock) 
    [![Last Commit](https://img.shields.io/github/last-commit/carbonsilicon-ai/CarsiDock?style=for-the-badge&logo=github)](https://github.com/carbonsilicon-ai/CarsiDock) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:16-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1039/D3SC05552C) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-7.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1039/D3SC05552C)



- **DeepBindBC**: A practical deep learning method for identifying native-like protein-ligand complexes in virtual screening (online).  

    [![Publication](https://img.shields.io/badge/Publication-Citations:18-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1016/j.ymeth.2022.07.009) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.20-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1016/j.ymeth.2022.07.009)



- **DeltaDock**: DeltaDock: A Unified Framework for Accurate, Efficient, and Physically Reliable Molecular Docking  
    [![Code](https://img.shields.io/github/stars/jiaxianyan/DeltaDock?style=for-the-badge&logo=github)](https://github.com/jiaxianyan/DeltaDock) 
    [![Last Commit](https://img.shields.io/github/last-commit/jiaxianyan/DeltaDock?style=for-the-badge&logo=github)](https://github.com/jiaxianyan/DeltaDock) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/bt) 



- **DiffBindFR**: DiffBindFR: an SE(3) equivariant network for flexible protein–ligand docking  
    [![Code](https://img.shields.io/github/stars/HBioquant/DiffBindFR?style=for-the-badge&logo=github)](https://github.com/HBioquant/DiffBindFR) 
    [![Last Commit](https://img.shields.io/github/last-commit/HBioquant/DiffBindFR?style=for-the-badge&logo=github)](https://github.com/HBioquant/DiffBindFR) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:6-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1039/D3SC06803J) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-7.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1039/D3SC06803J)



- **DiffDock**: A state-of-the-art method for molecular docking, incorporating diffusion steps and a significant improvement in performance and generalization capacity.  
    [![Code](https://img.shields.io/github/stars/gcorso/DiffDock?style=for-the-badge&logo=github)](https://github.com/gcorso/DiffDock) 
    [![Last Commit](https://img.shields.io/github/last-commit/gcorso/DiffDock?style=for-the-badge&logo=github)](https://github.com/gcorso/DiffDock) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/med/9780190607555.003.0020) 



- **DiffDock-NMDN**: Normalized Protein–Ligand Distance Likelihood Score for End-to-End Blind Docking and Virtual Screening  
    [![Code](https://img.shields.io/github/stars/SongXia-NYU/DiffDock-NMDN?style=for-the-badge&logo=github)](https://github.com/SongXia-NYU/DiffDock-NMDN) 
    [![Last Commit](https://img.shields.io/github/last-commit/SongXia-NYU/DiffDock-NMDN?style=for-the-badge&logo=github)](https://github.com/SongXia-NYU/DiffDock-NMDN) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.4c01014) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jcim.4c01014)



- **DiffDock-Pocket**:   
    [![Code](https://img.shields.io/github/stars/plainerman/DiffDock-Pocket?style=for-the-badge&logo=github)](https://github.com/plainerman/DiffDock-Pocket) 
    [![Last Commit](https://img.shields.io/github/last-commit/plainerman/DiffDock-Pocket?style=for-the-badge&logo=github)](https://github.com/plainerman/DiffDock-Pocket) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](None) 



- **EDM-Dock**: Efficient and accurate large library ligand docking with KarmaDock  
    [![Code](https://img.shields.io/github/stars/MatthewMasters/EDM-Dock?style=for-the-badge&logo=github)](https://github.com/MatthewMasters/EDM-Dock) 
    [![Last Commit](https://img.shields.io/github/last-commit/MatthewMasters/EDM-Dock?style=for-the-badge&logo=github)](https://github.com/MatthewMasters/EDM-Dock) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:57-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038/s43588-023-00511-5) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-12.00-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1038/s43588-023-00511-5)



- **ESF - scalar fields**: Implements Equivariant Scalar Fields for Molecular Docking with Fast Fourier Transforms, a machine learning-based ligand pose scoring function for rapid optimization.  
    [![Code](https://img.shields.io/github/stars/bjing2016/scalar-fields?style=for-the-badge&logo=github)](https://github.com/bjing2016/scalar-fields) 
    [![Last Commit](https://img.shields.io/github/last-commit/bjing2016/scalar-fields?style=for-the-badge&logo=github)](https://github.com/bjing2016/scalar-fields) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:1-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1109/9780470544792.ch3) 



- **EViS**: EViS is an enhanced virtual screening method integrating ligand docking, protein pocket template searching, and ligand template shape similarity calculations, utilizing a novel PL-score for evaluation.  
    [![Code](https://img.shields.io/github/stars/JingHuangLab/EViS?style=for-the-badge&logo=github)](https://github.com/JingHuangLab/EViS) 
    [![Last Commit](https://img.shields.io/github/last-commit/JingHuangLab/EViS?style=for-the-badge&logo=github)](https://github.com/JingHuangLab/EViS) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:9-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.1c00944) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jcim.1c00944)



- **FABind**: FABind: Fast and Accurate Protein-Ligand Binding  
    [![Code](https://img.shields.io/github/stars/QizhiPei/FABind?style=for-the-badge&logo=github)](https://github.com/QizhiPei/FABind) 
    [![Last Commit](https://img.shields.io/github/last-commit/QizhiPei/FABind?style=for-the-badge&logo=github)](https://github.com/QizhiPei/FABind) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.48550/arXiv.2310.06763) 



- **FABind+**: FABind: Fast and Accurate Protein-Ligand Binding  
    [![Code](https://img.shields.io/github/stars/QizhiPei/FABind?style=for-the-badge&logo=github)](https://github.com/QizhiPei/FABind) 
    [![Last Commit](https://img.shields.io/github/last-commit/QizhiPei/FABind?style=for-the-badge&logo=github)](https://github.com/QizhiPei/FABind) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.48550/arXiv.2310.06763) 



- **FeatureDock**: Protein-Ligand Docking Guided by Physicochemical Feature-Based Local Environment Learning using Transformer  
    [![Code](https://img.shields.io/github/stars/xuhuihuang/featuredock?style=for-the-badge&logo=github)](https://github.com/xuhuihuang/featuredock) 
    [![Last Commit](https://img.shields.io/github/last-commit/xuhuihuang/featuredock?style=for-the-badge&logo=github)](https://github.com/xuhuihuang/featuredock) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.26434/chemrxiv-2024-dh2rw) 



- **FlexPose**: Equivariant Flexible Modeling of the Protein–Ligand Binding Pose with Geometric Deep Learning  
    [![Code](https://img.shields.io/github/stars/tiejundong/FlexPose?style=for-the-badge&logo=github)](https://github.com/tiejundong/FlexPose) 
    [![Last Commit](https://img.shields.io/github/last-commit/tiejundong/FlexPose?style=for-the-badge&logo=github)](https://github.com/tiejundong/FlexPose) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:14-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jctc.3c00273) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.70-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jctc.3c00273)



- **GAABind**: GAABind is a Geometry-Aware Attention-Based Network for accurate protein-ligand binding pose and binding affinity prediction, featuring a comprehensive environment setup and dataset processing guide.  
    [![Code](https://img.shields.io/github/stars/Mercuryhs/GAABind/blob/main/README.MD?style=for-the-badge&logo=github)](https://github.com/Mercuryhs/GAABind/blob/main/README.MD) 
    [![Last Commit](https://img.shields.io/github/last-commit/Mercuryhs/GAABind/blob/main/README.MD?style=for-the-badge&logo=github)](https://github.com/Mercuryhs/GAABind/blob/main/README.MD) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:1-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bib/bbad462) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-6.80-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/bib/bbad462)



- **GNINA**: GNINA is a molecular docking program that incorporates scoring and optimization of ligands using convolutional neural networks, aiming to combine the versatility of smina and AutoDock Vina with the predictive power of deep learning.  
    [![Code](https://img.shields.io/github/stars/gnina/gnina?style=for-the-badge&logo=github)](https://github.com/gnina/gnina) 
    [![Last Commit](https://img.shields.io/github/last-commit/gnina/gnina?style=for-the-badge&logo=github)](https://github.com/gnina/gnina) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:333-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1186/s13321-021-00522-2) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-7.10-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1186/s13321-021-00522-2)



- **GNINA_KD**: Condensing Molecular Docking CNNs via Knowledge Distillation  
    [![Code](https://img.shields.io/github/stars/YanjingLiLi/GNINA_Knowledge_Distillation?style=for-the-badge&logo=github)](https://github.com/YanjingLiLi/GNINA_Knowledge_Distillation) 
    [![Last Commit](https://img.shields.io/github/last-commit/YanjingLiLi/GNINA_Knowledge_Distillation?style=for-the-badge&logo=github)](https://github.com/YanjingLiLi/GNINA_Knowledge_Distillation) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.26434/chemrxiv-2024-0jh8g) 



- **GalaxyDock-DL**: Protein–Ligand Docking by Global Optimization and Neural Network Energy  
    [![Code](https://img.shields.io/github/stars/seoklab/galaxydock_dl?style=for-the-badge&logo=github)](https://github.com/seoklab/galaxydock_dl) 
    [![Last Commit](https://img.shields.io/github/last-commit/seoklab/galaxydock_dl?style=for-the-badge&logo=github)](https://github.com/seoklab/galaxydock_dl) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:3-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jctc.4c00385) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.70-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jctc.4c00385)



- **Interformer**: interaction-aware model for protein-ligand docking and affinity prediction  
    [![Code](https://img.shields.io/github/stars/tencent-ailab/Interformer?style=for-the-badge&logo=github)](https://github.com/tencent-ailab/Interformer) 
    [![Last Commit](https://img.shields.io/github/last-commit/tencent-ailab/Interformer?style=for-the-badge&logo=github)](https://github.com/tencent-ailab/Interformer) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038/s41467-024-54440-6) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-14.70-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1038/s41467-024-54440-6)



- **KarmaDock**: Efficient and accurate large library ligand docking with KarmaDock  
    [![Code](https://img.shields.io/github/stars/schrojunzhang/KarmaDock/blob/main/README.md?style=for-the-badge&logo=github)](https://github.com/schrojunzhang/KarmaDock/blob/main/README.md) 
    [![Last Commit](https://img.shields.io/github/last-commit/schrojunzhang/KarmaDock/blob/main/README.md?style=for-the-badge&logo=github)](https://github.com/schrojunzhang/KarmaDock/blob/main/README.md) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:57-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038/s43588-023-00511-5) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-12.00-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1038/s43588-023-00511-5)



- **LigPose**: One-step Structure Prediction and Screening for Protein-Ligand Complexes using Multi-Task Geometric Deep Learning  
    [![Code](https://img.shields.io/badge/Code-Repository-blue?style=for-the-badge)](https://gitfront.io/r/LigPose/kMWuV4DW6JpE/LigPose4Review/) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.21203/rs.3.rs-1454132) 



- **PGBind**: pocket-guided explicit attention learning for protein–ligand docking  
    [![Code](https://img.shields.io/github/stars/ShenAoAO/PGBind?style=for-the-badge&logo=github)](https://github.com/ShenAoAO/PGBind) 
    [![Last Commit](https://img.shields.io/github/last-commit/ShenAoAO/PGBind?style=for-the-badge&logo=github)](https://github.com/ShenAoAO/PGBind) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:1-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bib/bbae455) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-6.80-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/bib/bbae455)



- **PLANTAIN**: Predicting LigANd pose wiTh an AI scoring functioN  
    [![Code](https://img.shields.io/github/stars/molecularmodelinglab/plantain?style=for-the-badge&logo=github)](https://github.com/molecularmodelinglab/plantain) 
    [![Last Commit](https://img.shields.io/github/last-commit/molecularmodelinglab/plantain?style=for-the-badge&logo=github)](https://github.com/molecularmodelinglab/plantain) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](None) 



- **PPDock**: Pocket Prediction-Based Protein–Ligand Blind Docking  
    [![Code](https://img.shields.io/github/stars/JieDuTQS/PPDock?style=for-the-badge&logo=github)](https://github.com/JieDuTQS/PPDock) 
    [![Last Commit](https://img.shields.io/github/last-commit/JieDuTQS/PPDock?style=for-the-badge&logo=github)](https://github.com/JieDuTQS/PPDock) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.4c01373) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jcim.4c01373)



- **PointVS**: SE(3)-equivariant point cloud networks designed for virtual screening, enabling E(3)-invariant predictions of binding pose and affinity using networks based on the EGNN graph neural network layer.  
    [![Code](https://img.shields.io/github/stars/jscant/PointVS?style=for-the-badge&logo=github)](https://github.com/jscant/PointVS) 
    [![Last Commit](https://img.shields.io/github/last-commit/jscant/PointVS?style=for-the-badge&logo=github)](https://github.com/jscant/PointVS) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.3c00322.s001) 



- **QuickBind**: QuickBind: A Light-Weight And Interpretable Molecular Docking Model  
    [![Code](https://img.shields.io/github/stars/aqlaboratory/QuickBind?style=for-the-badge&logo=github)](https://github.com/aqlaboratory/QuickBind) 
    [![Last Commit](https://img.shields.io/github/last-commit/aqlaboratory/QuickBind?style=for-the-badge&logo=github)](https://github.com/aqlaboratory/QuickBind) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.48550/arXiv.2410.16474) 



- **RapidDock**: RapidDock: Unlocking Proteome-scale Molecular Docking  

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.48550/arXiv.2411.00004) 



- **Re-Dock**: Re-Dock: Towards Flexible and Realistic Molecular Docking with Diffusion Bridge  

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.48550/arXiv.2402.11459) 



- **SurfDock**: Surface-Informed Diffusion Generative Model for Reliable and Accurate Protein-ligand Complex Prediction  
    [![Code](https://img.shields.io/github/stars/CAODH/SurfDock?style=for-the-badge&logo=github)](https://github.com/CAODH/SurfDock) 
    [![Last Commit](https://img.shields.io/github/last-commit/CAODH/SurfDock?style=for-the-badge&logo=github)](https://github.com/CAODH/SurfDock) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:1-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1101/2023.12.13.571408) 



- **TankBind**: Trigonometry-Aware Neural NetworKs for Drug-Protein Binding Structure Prediction  
    [![Code](https://img.shields.io/github/stars/luwei0917/TankBind?style=for-the-badge&logo=github)](https://github.com/luwei0917/TankBind) 
    [![Last Commit](https://img.shields.io/github/last-commit/luwei0917/TankBind?style=for-the-badge&logo=github)](https://github.com/luwei0917/TankBind) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:66-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1101/2022.06.06.495043) 



- **TopoFormer**: A topological transformer for protein-ligand complex interaction prediction, integrating multiscale topology techniques with a structure-to-sequence transformer model.  
    [![Code](https://img.shields.io/github/stars/WeilabMSU/TopoFormer?style=for-the-badge&logo=github)](https://github.com/WeilabMSU/TopoFormer) 
    [![Last Commit](https://img.shields.io/github/last-commit/WeilabMSU/TopoFormer?style=for-the-badge&logo=github)](https://github.com/WeilabMSU/TopoFormer) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.5281/zenodo.10892800) 



- **Uni-Dock**: GPU-Accelerated Docking Enables Ultralarge Virtual Screening (standalone).  
    [![Code](https://img.shields.io/github/stars/dptech-corp/Uni-Dock?style=for-the-badge&logo=github)](https://github.com/dptech-corp/Uni-Dock) 
    [![Last Commit](https://img.shields.io/github/last-commit/dptech-corp/Uni-Dock?style=for-the-badge&logo=github)](https://github.com/dptech-corp/Uni-Dock) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:35-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jctc.2c01145) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.70-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jctc.2c01145)



- **Uni-Mol**: A Universal 3D Molecular Representation Learning Framework  
    [![Code](https://img.shields.io/github/stars/deepmodeling/Uni-Mol?style=for-the-badge&logo=github)](https://github.com/deepmodeling/Uni-Mol) 
    [![Last Commit](https://img.shields.io/github/last-commit/deepmodeling/Uni-Mol?style=for-the-badge&logo=github)](https://github.com/deepmodeling/Uni-Mol) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:10-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.26434/chemrxiv-2022-jjm0j-v3) 



- **Uni-Mol v2**:   
    [![Code](https://img.shields.io/github/stars/deepmodeling/Uni-Mol?style=for-the-badge&logo=github)](https://github.com/deepmodeling/Uni-Mol) 
    [![Last Commit](https://img.shields.io/github/last-commit/deepmodeling/Uni-Mol?style=for-the-badge&logo=github)](https://github.com/deepmodeling/Uni-Mol) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:10-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.26434/chemrxiv-2022-jjm0j-v3) 



- **gnina-torch**: A PyTorch implementation of the GNINA molecular docking scoring function, designed for enhanced performance and adaptability.  
    [![Code](https://img.shields.io/github/stars/RMeli/gnina-torch/tree/0.0.2?tab=readme-ov-file?style=for-the-badge&logo=github)](https://github.com/RMeli/gnina-torch/tree/0.0.2?tab=readme-ov-file) 
    [![Last Commit](https://img.shields.io/github/last-commit/RMeli/gnina-torch/tree/0.0.2?tab=readme-ov-file?style=for-the-badge&logo=github)](https://github.com/RMeli/gnina-torch/tree/0.0.2?tab=readme-ov-file) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.5281/zenodo.6943066) 



- **vScreenML**: A machine learning classifier designed for virtual screening, allowing for the rescoring of hits to eliminate false positives, based on the Dataset of Congruent Inhibitors and Decoys (D-COID).  
    [![Code](https://img.shields.io/github/stars/karanicolaslab/vScreenML?style=for-the-badge&logo=github)](https://github.com/karanicolaslab/vScreenML) 
    [![Last Commit](https://img.shields.io/github/last-commit/karanicolaslab/vScreenML?style=for-the-badge&logo=github)](https://github.com/karanicolaslab/vScreenML) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:147-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1073/pnas.2000585117) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-9.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1073/pnas.2000585117)



- **vScreenML2.0**: Improved Machine Learning Classification for Reducing False Positives in Structure-Based Virtual Screening  
    [![Code](https://img.shields.io/github/stars/gandrianov/vScreenML2?style=for-the-badge&logo=github)](https://github.com/gandrianov/vScreenML2) 
    [![Last Commit](https://img.shields.io/github/last-commit/gandrianov/vScreenML2?style=for-the-badge&logo=github)](https://github.com/gandrianov/vScreenML2) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.3390/ijms252212350) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.90-purple?style=for-the-badge&logo=academia)](https://doi.org/10.3390/ijms252212350)


### **Flexible**

- **CarsiDock-Flex**:   
    [![Code](https://img.shields.io/github/stars/xqh19970407/CarsiInduce?style=for-the-badge&logo=github)](https://github.com/xqh19970407/CarsiInduce) 
    [![Last Commit](https://img.shields.io/github/last-commit/xqh19970407/CarsiInduce?style=for-the-badge&logo=github)](https://github.com/xqh19970407/CarsiInduce) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jmedchem.4c02740) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-6.80-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jmedchem.4c02740)


### **Reviews**

- **Benchmarking AI-powered docking methods from the perspective of virtual screening**:   

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038/s42256-025-00993-0) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-18.80-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1038/s42256-025-00993-0)


## **MetalloProteins**


- **MetalDock**: MetalDock: An Open Access Docking Tool for Easy and Reproducible Docking of Metal Complexes  
    [![Code](https://img.shields.io/github/stars/MatthijsHak/MetalDock?style=for-the-badge&logo=github)](https://github.com/MatthijsHak/MetalDock) 
    [![Last Commit](https://img.shields.io/github/last-commit/MatthijsHak/MetalDock?style=for-the-badge&logo=github)](https://github.com/MatthijsHak/MetalDock) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:14-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.3c01582) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jcim.3c01582)


## **Multi-Ligand**


- **HARMONICFlow**: Harmonic Self-Conditioned Flow Matching for Multi-Ligand Docking and Binding Site Design  
    [![Code](https://img.shields.io/github/stars/HannesStark/FlowSite?style=for-the-badge&logo=github)](https://github.com/HannesStark/FlowSite) 
    [![Last Commit](https://img.shields.io/github/last-commit/HannesStark/FlowSite?style=for-the-badge&logo=github)](https://github.com/HannesStark/FlowSite) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.48550/arXiv.2310.05764) 


## **Negative Image based**


- **O-LAP**: Building shape-focused pharmacophore models for effective docking screening  
    [![Code](https://img.shields.io/github/stars/jvlehtonen/overlap-toolkit?style=for-the-badge&logo=github)](https://github.com/jvlehtonen/overlap-toolkit) 
    [![Last Commit](https://img.shields.io/github/last-commit/jvlehtonen/overlap-toolkit?style=for-the-badge&logo=github)](https://github.com/jvlehtonen/overlap-toolkit) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:1-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1186/s13321-024-00857-6) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-7.10-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1186/s13321-024-00857-6)



- **PANTHER**: Negative Image based docking and scoring  

    [![Publication](https://img.shields.io/badge/Publication-Citations:33-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1007/s10822-015-9870-3) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-3.00-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1007/s10822-015-9870-3)


## **Pose Optimisation**


- **DeepRMSD-Vina**: DeepRMSD+Vina is a computational framework integrating ligand binding pose optimization and screening, utilizing deep learning alongside the classical Vina scoring function.  
    [![Code](https://img.shields.io/github/stars/zchwang/DeepRMSD-Vina_Optimization?style=for-the-badge&logo=github)](https://github.com/zchwang/DeepRMSD-Vina_Optimization) 
    [![Last Commit](https://img.shields.io/github/last-commit/zchwang/DeepRMSD-Vina_Optimization?style=for-the-badge&logo=github)](https://github.com/zchwang/DeepRMSD-Vina_Optimization) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:32-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bib/bbac520) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-6.80-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/bib/bbac520)


## **Protein Docking**


- **EquiDock**: EquiDock employs geometric deep learning for fast and accurate rigid 3D protein-protein docking, focusing on efficiency and accessibility with comprehensive preprocessing and training guidelines.  
    [![Code](https://img.shields.io/github/stars/octavian-ganea/equidock_public?style=for-the-badge&logo=github)](https://github.com/octavian-ganea/equidock_public) 
    [![Last Commit](https://img.shields.io/github/last-commit/octavian-ganea/equidock_public?style=for-the-badge&logo=github)](https://github.com/octavian-ganea/equidock_public) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:4-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1101/2021.06.06.447297) 



- **LightDock**: The open-source macromolecular docking framework written in Python  
    [![Code](https://img.shields.io/github/stars/lightdock/lightdock?style=for-the-badge&logo=github)](https://github.com/lightdock/lightdock) 
    [![Last Commit](https://img.shields.io/github/last-commit/lightdock/lightdock?style=for-the-badge&logo=github)](https://github.com/lightdock/lightdock) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:94-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btx555) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.40-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/bioinformatics/btx555)



- **PyDock3**: Electrostatics and desolvation for effective scoring of rigid-body protein-protein docking  
    [![Code](https://img.shields.io/github/stars/brianjimenez/pydock_tutorial?style=for-the-badge&logo=github)](https://github.com/brianjimenez/pydock_tutorial) 
    [![Last Commit](https://img.shields.io/github/last-commit/brianjimenez/pydock_tutorial?style=for-the-badge&logo=github)](https://github.com/brianjimenez/pydock_tutorial) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:293-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1002/prot.21419) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-3.20-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1002/prot.21419)


## **Quantum**


- **Zhang et al.**: Encoding Molecular Docking for Quantum Computers  

    [![Publication](https://img.shields.io/badge/Publication-Citations:6-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jctc.3c00943) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.70-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jctc.3c00943)


## **Reviews**


- **Revolutionizing drug discovery: an AI-powered transformation of molecular docking**: Revolutionizing drug discovery: an AI-powered transformation of molecular docking  

    [![Publication](https://img.shields.io/badge/Publication-Citations:3-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1007/s00044-024-03253-9) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-2.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1007/s00044-024-03253-9)


## **Shape-based**


- **PheSA**: An Open-Source Tool for Pharmacophore-Enhanced Shape Alignment  
    [![Code](https://img.shields.io/github/stars/joewah/PheSAExamples?style=for-the-badge&logo=github)](https://github.com/joewah/PheSAExamples) 
    [![Last Commit](https://img.shields.io/github/last-commit/joewah/PheSAExamples?style=for-the-badge&logo=github)](https://github.com/joewah/PheSAExamples) 

    [![Publication](https://img.shields.io/badge/Publication-Citations:1-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.4c00516) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-5.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1021/acs.jcim.4c00516)


## **Template**


- **FitDock**: fits initial conformation to the given template using a hierachical multi-feature alignment approach, subsequently explores the possible conformations, and finally outputs refined docking poses  

    [![Publication](https://img.shields.io/badge/Publication-Citations:107-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bib/bbac087) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-6.80-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/bib/bbac087)


## **Water**


- **WatVina**: Watvina facilitates drug design with support for explicit or implicit waters, pharmacophore, or position-constrained docking, and external torsion parameters, enhancing the Autodock Vina engine.  
    [![Code](https://img.shields.io/github/stars/biocheming/watvina?style=for-the-badge&logo=github)](https://github.com/biocheming/watvina) 
    [![Last Commit](https://img.shields.io/github/last-commit/biocheming/watvina?style=for-the-badge&logo=github)](https://github.com/biocheming/watvina) 



## **WebServers**


- **MolModa**: accessible and secure molecular docking in a web browser.  

    [![Publication](https://img.shields.io/badge/Publication-Citations:2-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/nar/gkae406) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-16.60-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1093/nar/gkae406)



- **TOMEv3**: @TOME 3.0: Interfacing Protein Structure Modeling and Ligand Docking  

    [![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1016/j.jmb.2024.168704) 
    [![JIF](https://img.shields.io/badge/Impact_Factor-4.70-purple?style=for-the-badge&logo=academia)](https://doi.org/10.1016/j.jmb.2024.168704)

    [![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://atome.cbs.cnrs.fr/ATOME_V3/index.html) 

