
## **All-Atom-DL methods**
- **AlphaFold3**:   
	[![Publication](https://img.shields.io/badge/Publication-Citations:437-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038/s41586-024-07487-w)  
	[![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://golgi.sandbox.google.com/)  
- **RoseTTAFold-AllAtom**: RoseTTAFold All-Atom (RFAA) is a deep network capable of modeling full biological assemblies containing proteins, nucleic acids, small molecules, metals, and covalent modifications with high accuracy.  
	[![Publication](https://img.shields.io/badge/Publication-Citations:86-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1126/science.adl2528)  
	[![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://www.biorxiv.org/content/10.1101/2023.10.09.561603v1)  
- **DynamicBind**: DynamicBind recovers ligand-specific conformations from unbound protein structures (e.g. AF2-predicted structures), promoting efficient transitions between different equilibrium states.  
	[![Code](https://img.shields.io/github/stars/luwei0917/DynamicBind?style=for-the-badge&logo=github)](https://github.com/luwei0917/DynamicBind)  
	[![Last Commit](https://img.shields.io/github/last-commit/luwei0917/DynamicBind?style=for-the-badge&logo=github)](https://github.com/luwei0917/DynamicBind)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:15-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038/s41467-024-45461-2)  
- **DiffusionProteinLigand**: End-to-end protein–ligand complex structure generation with diffusion-based generative models (standalone).  
	[![Code](https://img.shields.io/github/stars/shuyana/DiffusionProteinLigand?style=for-the-badge&logo=github)](https://github.com/shuyana/DiffusionProteinLigand)  
	[![Last Commit](https://img.shields.io/github/last-commit/shuyana/DiffusionProteinLigand?style=for-the-badge&logo=github)](https://github.com/shuyana/DiffusionProteinLigand)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:12-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1186/s12859-023-05354-5)  
- **NeuralPlexer**: deep generative model to jointly predict protein-ligand complex 3D structures and beyond.  
	[![Code](https://img.shields.io/github/stars/zrqiao/NeuralPLexer?style=for-the-badge&logo=github)](https://github.com/zrqiao/NeuralPLexer)  
	[![Last Commit](https://img.shields.io/github/last-commit/zrqiao/NeuralPLexer?style=for-the-badge&logo=github)](https://github.com/zrqiao/NeuralPLexer)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:17-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038/s42256-024-00792-z)  

## **Allosteric sites**
- **FASTDock**: FASTDock is a pipeline for allosteric drug discovery, offering scripts and a Jupyter notebook for efficiently generating and analyzing docking grids, clusters, and fingerprint screenings.  
	[![Code](https://img.shields.io/github/stars/BrooksResearchGroup-UM/FASTDock?style=for-the-badge&logo=github)](https://github.com/BrooksResearchGroup-UM/FASTDock)  
	[![Last Commit](https://img.shields.io/github/last-commit/BrooksResearchGroup-UM/FASTDock?style=for-the-badge&logo=github)](https://github.com/BrooksResearchGroup-UM/FASTDock)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=arxiv)](https://doi.org/10.26434/chemrxiv-2023-7c43h)  

## **Blind Docking**
- **CBDock2**: CBDock2 is an improved protein-ligand blind docking tool integrating cavity detection, docking, and homologous template fitting to suggest novel therapeutic targets for biological and pharmaceutical studies.  
	[![Publication](https://img.shields.io/badge/Publication-Citations:357-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/nar/gkac394)  
	[![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://cadd.labshare.cn/cb-dock2/php/index.php)  
- **CoBDock**: CoBDock is a reference implementation of the COBDock algorithm, detailing steps for setup and execution on Linux, with a focus on integrating various molecular docking and pocket identification algorithms.  
	[![Code](https://img.shields.io/github/stars/DavidMcDonald1993/cobdock?style=for-the-badge&logo=github)](https://github.com/DavidMcDonald1993/cobdock)  
	[![Last Commit](https://img.shields.io/github/last-commit/DavidMcDonald1993/cobdock?style=for-the-badge&logo=github)](https://github.com/DavidMcDonald1993/cobdock)  

## **Classical**
- **AutoDock Vina**:   
	[![Code](https://img.shields.io/github/stars/ccsb-scripps/AutoDock-Vina?style=for-the-badge&logo=github)](https://github.com/ccsb-scripps/AutoDock-Vina)  
	[![Last Commit](https://img.shields.io/github/last-commit/ccsb-scripps/AutoDock-Vina?style=for-the-badge&logo=github)](https://github.com/ccsb-scripps/AutoDock-Vina)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:2163-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.1c00203)  
	[![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://autodock-vina.readthedocs.io/en/latest/index.html)  
- **BR-NiB**: Brute Force Negative Image-Based Optimization. A docking rescoring method that relyes on shape/ electrostatic potential similarity between the docking poses of ligands and the cavity-based negative images (standalone).  
	[![Code](https://img.shields.io/github/stars/jvlehtonen/brutenib?style=for-the-badge&logo=github)](https://github.com/jvlehtonen/brutenib)  
	[![Last Commit](https://img.shields.io/github/last-commit/jvlehtonen/brutenib?style=for-the-badge&logo=github)](https://github.com/jvlehtonen/brutenib)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:3-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.1c01145)  
- **DockThor**:   
	[![Publication](https://img.shields.io/badge/Publication-Citations:N/A-blue?style=for-the-badge&logo=bookstack)](https://www.nature.com/articles/s41598-021-84700-0)  
	[![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://dockthor.lncc.br/v2/)  
- **FWAVina**: The content for FWAVina was not accessible due to restrictions or an error from the URL provided.  
	[![Code](https://img.shields.io/github/stars/eddyblue/FWAVina?style=for-the-badge&logo=github)](https://github.com/eddyblue/FWAVina/tree/master)  
	[![Last Commit](https://img.shields.io/github/last-commit/eddyblue/FWAVina?style=for-the-badge&logo=github)](https://github.com/eddyblue/FWAVina/tree/master)  
- **GeauxDock**: An ultra-fast automated docking program from LSU, predicting how small ligands bind to macromolecules using a novel hybrid force field and a Monte Carlo protocol.  
	[![Code](https://img.shields.io/badge/Code)](http://www.brylinski.org/geauxdock)
	[![Publication](https://img.shields.io/badge/Publication-Citations:24-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1371/journal.pone.0158898)  
	[![Link](https://img.shields.io/badge/Link-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](https://www.brylinski.org/geauxdock)  
- **GLOW-IVES**: Provides Python implementation of GLOW (auGmented sampLing with softened vdW potential) and IVES (Iterative Ensemble Sampling) protocols for pose sampling, along with new cross-docking datasets.  
	[![Code](https://img.shields.io/github/stars/drorlab/GLOW_IVES?style=for-the-badge&logo=github)](https://github.com/drorlab/GLOW_IVES)  
	[![Last Commit](https://img.shields.io/github/last-commit/drorlab/GLOW_IVES?style=for-the-badge&logo=github)](https://github.com/drorlab/GLOW_IVES)  
- **HESS**:   
	[![Code](https://img.shields.io/github/stars/Entroforce/Hess?style=for-the-badge&logo=github)](https://github.com/Entroforce/Hess)  
	[![Last Commit](https://img.shields.io/github/last-commit/Entroforce/Hess?style=for-the-badge&logo=github)](https://github.com/Entroforce/Hess)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=arxiv)](https://doi.org/10.26434/chemrxiv-2023-jh1kk)  
- **JAMDA**:   
	[![Publication](https://img.shields.io/badge/Publication-Citations:4-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.3c01573)  
	[![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://proteins.plus/)  
- **labodock**: LABODOCK offers a collection of Jupyter Notebooks for molecular docking on Google Colab with minimal coding, streamlining pre- and post-docking processes.  
	[![Code](https://img.shields.io/github/stars/RyanZR/labodock?style=for-the-badge&logo=github)](https://github.com/RyanZR/labodock)  
	[![Last Commit](https://img.shields.io/github/last-commit/RyanZR/labodock?style=for-the-badge&logo=github)](https://github.com/RyanZR/labodock)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.5281/zenodo.8246977-blue)  
- **MzDock**:   
	[![Code](https://img.shields.io/github/stars/Muzatheking12/MzDOCK?style=for-the-badge&logo=github)](https://github.com/Muzatheking12/MzDOCK)  
	[![Last Commit](https://img.shields.io/github/last-commit/Muzatheking12/MzDOCK?style=for-the-badge&logo=github)](https://github.com/Muzatheking12/MzDOCK)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1002/jcc.27390)  
- **parallel-PLANTS**: Offers a method for parallel molecular docking using the PLANTS software, aimed at academic use.  
	[![Code](https://img.shields.io/github/stars/discoverdata/parallel-PLANTS?style=for-the-badge&logo=github)](https://github.com/discoverdata/parallel-PLANTS)  
	[![Last Commit](https://img.shields.io/github/last-commit/discoverdata/parallel-PLANTS?style=for-the-badge&logo=github)](https://github.com/discoverdata/parallel-PLANTS)  
- **ProBiS-Dock**: A Hybrid Multitemplate Homology Flexible Docking Algorithm Enabled by Protein Binding Site Comparison (standalone).  
	[![Code](https://img.shields.io/badge/Code)](https://gitlab.com/janezkonc/insidrug/-/releases/probis_dock_algorithm_v1.0.0)
	[![Publication](https://img.shields.io/badge/Publication-Citations:5-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.1c01176)  
	[![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://insilab.org/probisdock)  
- **QVINA**: QuickVina 2 aims to accurately speed up AutoDock Vina, providing up to 20.49-fold acceleration with high correlation in binding energy prediction.  
	[![Code](https://img.shields.io/github/stars/QVina/qvina?style=for-the-badge&logo=github)](https://github.com/QVina/qvina)  
	[![Last Commit](https://img.shields.io/github/last-commit/QVina/qvina?style=for-the-badge&logo=github)](https://github.com/QVina/qvina)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:196-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btv082)  
- **RDPSO_Vina**: A fast docking tool utilizing random drift particle swarm optimization based on the AutoDock Vina and PSOVina framework.  
	[![Code](https://img.shields.io/github/stars/li-jin-xing/RDPSOVina?style=for-the-badge&logo=github)](https://github.com/li-jin-xing/RDPSOVina)  
	[![Last Commit](https://img.shields.io/github/last-commit/li-jin-xing/RDPSOVina?style=for-the-badge&logo=github)](https://github.com/li-jin-xing/RDPSOVina)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:3-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1007/s10822-022-00455-4)  
- **restretto**: Effective Protein–Ligand Docking Strategy via Fragment Reuse and a Proof-of-Concept Implementation (standalone).  
	[![Code](https://img.shields.io/github/stars/akiyamalab/restretto?style=for-the-badge&logo=github)](https://github.com/akiyamalab/restretto)  
	[![Last Commit](https://img.shields.io/github/last-commit/akiyamalab/restretto?style=for-the-badge&logo=github)](https://github.com/akiyamalab/restretto)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:3-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acsomega.2c03470)  
- **RxDock**: a fast, versatile, and open-source program for docking ligands to proteins and nucleic acids (standalone).  
	[![Code](https://img.shields.io/github/stars/rxdock/rxdock?style=for-the-badge&logo=github)](https://github.com/rxdock/rxdock)  
	[![Last Commit](https://img.shields.io/github/last-commit/rxdock/rxdock?style=for-the-badge&logo=github)](https://github.com/rxdock/rxdock)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:412-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1371/journal.pcbi.1003571)  
- **SMINA**: A fork of AutoDock Vina that supports scoring function development and high-performance energy minimization, maintained by the University of Pittsburgh.  
	[![Code](https://img.shields.io/badge/Code)](https://sourceforge.net/projects/smina/)
	[![Publication](https://img.shields.io/badge/Publication-Citations:687-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/ci300604z)  
- **Uni-Dock**: Uni-Dock is a GPU-accelerated molecular docking program that supports various scoring functions and achieves significant speed-up compared with AutoDock Vina on a single CPU core.  
	[![Code](https://img.shields.io/github/stars/dptech-corp/Uni-Dock?style=for-the-badge&logo=github)](https://github.com/dptech-corp/Uni-Dock)  
	[![Last Commit](https://img.shields.io/github/last-commit/dptech-corp/Uni-Dock?style=for-the-badge&logo=github)](https://github.com/dptech-corp/Uni-Dock)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:22-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jctc.2c01145)  
- **VinaCarb**: The content for VinaCarb was not available from the URL provided.  
	[![Publication](https://img.shields.io/badge/Publication-Citations:93-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jctc.5b00834)  
	[![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://glycam.org/docs/othertoolsservice/download-docs/publication-materials/vina-carb/index.html)  
- **VinaGPU2.0**:   
	[![Code](https://img.shields.io/github/stars/DeltaGroupNJUPT/Vina-GPU-2.0?style=for-the-badge&logo=github)](https://github.com/DeltaGroupNJUPT/Vina-GPU-2.0)  
	[![Last Commit](https://img.shields.io/github/last-commit/DeltaGroupNJUPT/Vina-GPU-2.0?style=for-the-badge&logo=github)](https://github.com/DeltaGroupNJUPT/Vina-GPU-2.0)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:31-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.2c01504)  
- **VinaXB (halogen-bonding)**: Introduces a halogen bonding scoring function (XBSF) in AutoDock Vina, termed AutoDock VinaXB, to improve docking accuracy with halogenated ligands.  
	[![Code](https://img.shields.io/github/stars/sirimullalab/vinaXB?style=for-the-badge&logo=github)](https://github.com/sirimullalab/vinaXB/tree/master)  
	[![Last Commit](https://img.shields.io/github/last-commit/sirimullalab/vinaXB?style=for-the-badge&logo=github)](https://github.com/sirimullalab/vinaXB/tree/master)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:100-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1186/s13321-016-0139-1)  
	[![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-016-0139-1)  
- **Surflex-Tools**: starting with version 4 (standalone).  
	[![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://www.biopharmics.com/)  

## **Consensus**
- **dockECR**:   
	[![Code](https://img.shields.io/github/stars/rochoa85/dockECR?style=for-the-badge&logo=github)](https://github.com/rochoa85/dockECR)  
	[![Last Commit](https://img.shields.io/github/last-commit/rochoa85/dockECR?style=for-the-badge&logo=github)](https://github.com/rochoa85/dockECR)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:11-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1016/j.jmgm.2021.108023)  
- **DockingPie**: DockingPie is a PyMOL plugin that facilitates consensus docking and scoring analyses, integrating four docking programs (Smina, Autodock Vina, RxDock, and ADFR) to offer a versatile platform for molecular and consensus docking.  
	[![Code](https://img.shields.io/github/stars/paiardin/DockingPie?style=for-the-badge&logo=github)](https://github.com/paiardin/DockingPie)  
	[![Last Commit](https://img.shields.io/github/last-commit/paiardin/DockingPie?style=for-the-badge&logo=github)](https://github.com/paiardin/DockingPie)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:21-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btac452)  
- **Exponential Consensus Ranking**:   
	[![Publication](https://img.shields.io/badge/Publication-Citations:N/A-blue?style=for-the-badge&logo=bookstack)](https://www.nature.com/articles/s41598-019-41594-3#Sec8)  
- **VoteDock**:   
	[![Publication](https://img.shields.io/badge/Publication-Citations:73-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1002/jcc.21642)  

## **Flexible Docking**
- **ADFR**: AutoDockFR is a protein-ligand docking program supporting selective receptor flexibility and covalent docking, part of the ADFR suite for streamlined docking procedures.  
	[![Publication](https://img.shields.io/badge/Publication-Citations:335-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1371/journal.pcbi.1004586)  
	[![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://ccsb.scripps.edu/adfr/)  
- **DSDPFlex**:   
	[![Code](https://img.shields.io/github/stars/PKUGaoGroup/DSDP?style=for-the-badge&logo=github)](https://github.com/PKUGaoGroup/DSDP)  
	[![Last Commit](https://img.shields.io/github/last-commit/PKUGaoGroup/DSDP?style=for-the-badge&logo=github)](https://github.com/PKUGaoGroup/DSDP)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:1-blue?style=for-the-badge&logo=arxiv)](https://doi.org/10.26434/chemrxiv-2023-bcw0g-v2)  
	[![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://chemrxiv.org/engage/chemrxiv/article-details/6572d98429a13c4d47f6b4c6)  
- **FlexAID**:   
	[![Code](https://img.shields.io/github/stars/NRGlab/FlexAID?style=for-the-badge&logo=github)](https://github.com/NRGlab/FlexAID)  
	[![Last Commit](https://img.shields.io/github/last-commit/NRGlab/FlexAID?style=for-the-badge&logo=github)](https://github.com/NRGlab/FlexAID)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:50-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.5b00078)  
- **GNINA**: GNINA is a molecular docking program that incorporates scoring and optimization of ligands using convolutional neural networks, aiming to combine the versatility of smina and AutoDock Vina with the predictive power of deep learning.  
	[![Code](https://img.shields.io/github/stars/gnina/gnina?style=for-the-badge&logo=github)](https://github.com/gnina/gnina)  
	[![Last Commit](https://img.shields.io/github/last-commit/gnina/gnina?style=for-the-badge&logo=github)](https://github.com/gnina/gnina)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:228-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1186/s13321-021-00522-2)  
- **iDock**: iDock is a multithreaded virtual screening tool for flexible ligand docking in computational drug discovery, inspired by AutoDock Vina and hosted on GitHub under Apache License 2.0.  
	[![Code](https://img.shields.io/github/stars/gloglita/idock?style=for-the-badge&logo=github)](https://github.com/gloglita/idock)  
	[![Last Commit](https://img.shields.io/github/last-commit/gloglita/idock?style=for-the-badge&logo=github)](https://github.com/gloglita/idock)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:61-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1109/CIBCB.2012.6217214)  
- **PackDock**: Describes PackDock as a diffusion-based side chain packing model for flexible protein-ligand docking, indicating code will be available following the publication of their paper.  
	[![Code](https://img.shields.io/github/stars/Zhang-Runze/PackDock?style=for-the-badge&logo=github)](https://github.com/Zhang-Runze/PackDock)  
	[![Last Commit](https://img.shields.io/github/last-commit/Zhang-Runze/PackDock?style=for-the-badge&logo=github)](https://github.com/Zhang-Runze/PackDock)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1101/2024.01.31.578200)  
- **tiny_IFD**: Offers lightweight induced fit docking capabilities.  
	[![Code](https://img.shields.io/github/stars/darrenjhsu/tiny_IFD?style=for-the-badge&logo=github)](https://github.com/darrenjhsu/tiny_IFD)  
	[![Last Commit](https://img.shields.io/github/last-commit/darrenjhsu/tiny_IFD?style=for-the-badge&logo=github)](https://github.com/darrenjhsu/tiny_IFD)  
- **hybrid-SA-IFD**: Robust Induced Fit Docking Approach with the Combination ofthe Hybrid All-Atom/United-Atom/Coarse-Grained Model andSimulated Annealing  
	[![Code](https://img.shields.io/github/stars/ldx022/hybrid-SA-IFD?style=for-the-badge&logo=github)](https://github.com/ldx022/hybrid-SA-IFD)  
	[![Last Commit](https://img.shields.io/github/last-commit/ldx022/hybrid-SA-IFD?style=for-the-badge&logo=github)](https://github.com/ldx022/hybrid-SA-IFD)  

## **Fragment-based**
- **Spresso**: an ultrafast compound pre-screening method based on compound decomposition  
	[![Publication](https://img.shields.io/badge/Publication-Citations:13-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btx178)  
	[![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://www.bi.cs.titech.ac.jp/spresso/)  

## **HPC enabled**
- **POAP**:   
	[![Code](https://img.shields.io/github/stars/inpacdb/POAP?style=for-the-badge&logo=github)](https://github.com/inpacdb/POAP)  
	[![Last Commit](https://img.shields.io/github/last-commit/inpacdb/POAP?style=for-the-badge&logo=github)](https://github.com/inpacdb/POAP)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:62-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1016/j.compbiolchem.2018.02.012)  
- **VinaLC**:   
	[![Code](https://img.shields.io/github/stars/XiaohuaZhangLLNL/VinaLC?style=for-the-badge&logo=github)](https://github.com/XiaohuaZhangLLNL/VinaLC)  
	[![Last Commit](https://img.shields.io/github/last-commit/XiaohuaZhangLLNL/VinaLC?style=for-the-badge&logo=github)](https://github.com/XiaohuaZhangLLNL/VinaLC)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](http://onlinelibrary.wiley.com/doi/10.1002/jcc.23214/abstract)  
- **VinaMPI**:   
	[![Publication](https://img.shields.io/badge/Publication-Citations:62-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1002/jcc.23367)  
- **VinaSC**:   
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1109/BIBM.2016.7822624)  

## **MD-based**
- **ColDock**:   
	[![Publication](https://img.shields.io/badge/Publication-Citations:15-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jpcb.8b02756)  

## **ML-based**
- **ArtiDock**:   
	[![Publication](https://img.shields.io/badge/Publication-Citations:1-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1101/2024.03.14.585019)  
- **Re-Dock**:   
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.48550/arXiv.2402.11459)  
- **EDM-Dock**:   
	[![Code](https://img.shields.io/github/stars/MatthewMasters/EDM-Dock?style=for-the-badge&logo=github)](https://github.com/MatthewMasters/EDM-Dock)  
	[![Last Commit](https://img.shields.io/github/last-commit/MatthewMasters/EDM-Dock?style=for-the-badge&logo=github)](https://github.com/MatthewMasters/EDM-Dock)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:26-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038/s43588-023-00511-5)  
- **FlexPose**:   
	[![Code](https://img.shields.io/github/stars/tiejundong/FlexPose?style=for-the-badge&logo=github)](https://github.com/tiejundong/FlexPose)  
	[![Last Commit](https://img.shields.io/github/last-commit/tiejundong/FlexPose?style=for-the-badge&logo=github)](https://github.com/tiejundong/FlexPose)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:7-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jctc.3c00273)  
- **AQDNet**: Implements a Deep Neural Network for Protein-Ligand Docking Simulation, focusing on identifying correct binding poses through convolutional neural network approaches.  
	[![Code](https://img.shields.io/github/stars/koji11235/AQDnet?style=for-the-badge&logo=github)](https://github.com/koji11235/AQDnet)  
	[![Last Commit](https://img.shields.io/github/last-commit/koji11235/AQDnet?style=for-the-badge&logo=github)](https://github.com/koji11235/AQDnet)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:5-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acsomega.3c02411)  
- **CarsiDock**: a deep learning paradigm for accurate protein–ligand docking and screening based on large-scale pre-training (standalone, 2024).  
	[![Code](https://img.shields.io/github/stars/carbonsilicon-ai/CarsiDock?style=for-the-badge&logo=github)](https://github.com/carbonsilicon-ai/CarsiDock)  
	[![Last Commit](https://img.shields.io/github/last-commit/carbonsilicon-ai/CarsiDock?style=for-the-badge&logo=github)](https://github.com/carbonsilicon-ai/CarsiDock)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:6-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1039/D3SC05552C)  
- **DeepBindBC**: A practical deep learning method for identifying native-like protein-ligand complexes in virtual screening (online).  
	[![Publication](https://img.shields.io/badge/Publication-Citations:16-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1016/j.ymeth.2022.07.009)  
	[![Link](https://img.shields.io/badge/Link-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](http://cbblab.siat.ac.cn/DeepBindBC/index.php)  
- **DiffBindFR**:   
	[![Code](https://img.shields.io/github/stars/HBioquant/DiffBindFR?style=for-the-badge&logo=github)](https://github.com/HBioquant/DiffBindFR)  
	[![Last Commit](https://img.shields.io/github/last-commit/HBioquant/DiffBindFR?style=for-the-badge&logo=github)](https://github.com/HBioquant/DiffBindFR)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:2-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1039/D3SC06803J)  
- **DiffDock**: A state-of-the-art method for molecular docking, incorporating diffusion steps and a significant improvement in performance and generalization capacity.  
	[![Code](https://img.shields.io/github/stars/gcorso/DiffDock?style=for-the-badge&logo=github)](https://github.com/gcorso/DiffDock)  
	[![Last Commit](https://img.shields.io/github/last-commit/gcorso/DiffDock?style=for-the-badge&logo=github)](https://github.com/gcorso/DiffDock)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:N/A-blue?style=for-the-badge&logo=arxiv)](https://arxiv.org/abs/2210.01776)  
- **DiffDock-Pocket**:   
	[![Code](https://img.shields.io/github/stars/plainerman/DiffDock-Pocket?style=for-the-badge&logo=github)](https://github.com/plainerman/DiffDock-Pocket)  
	[![Last Commit](https://img.shields.io/github/last-commit/plainerman/DiffDock-Pocket?style=for-the-badge&logo=github)](https://github.com/plainerman/DiffDock-Pocket)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:N/A-blue?style=for-the-badge&logo=bookstack)](https://www.mlsb.io/papers_2023/DiffDock-Pocket_Diffusion_for_Pocket-Level_Docking_with_Sidechain_Flexibility.pdf)  
- **ESF - scalar fields**: Implements Equivariant Scalar Fields for Molecular Docking with Fast Fourier Transforms, a machine learning-based ligand pose scoring function for rapid optimization.  
	[![Code](https://img.shields.io/github/stars/bjing2016/scalar-fields?style=for-the-badge&logo=github)](https://github.com/bjing2016/scalar-fields)  
	[![Last Commit](https://img.shields.io/github/last-commit/bjing2016/scalar-fields?style=for-the-badge&logo=github)](https://github.com/bjing2016/scalar-fields)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:N/A-blue?style=for-the-badge&logo=arxiv)](https://arxiv.org/abs/2312.04323)  
- **EViS**: EViS is an enhanced virtual screening method integrating ligand docking, protein pocket template searching, and ligand template shape similarity calculations, utilizing a novel PL-score for evaluation.  
	[![Code](https://img.shields.io/github/stars/JingHuangLab/EViS?style=for-the-badge&logo=github)](https://github.com/JingHuangLab/EViS)  
	[![Last Commit](https://img.shields.io/github/last-commit/JingHuangLab/EViS?style=for-the-badge&logo=github)](https://github.com/JingHuangLab/EViS)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:6-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.1c00944)  
- **FABind**:   
	[![Code](https://img.shields.io/github/stars/QizhiPei/FABind?style=for-the-badge&logo=github)](https://github.com/QizhiPei/FABind)  
	[![Last Commit](https://img.shields.io/github/last-commit/QizhiPei/FABind?style=for-the-badge&logo=github)](https://github.com/QizhiPei/FABind)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:N/A-blue?style=for-the-badge&logo=arxiv)](https://arxiv.org/abs/2310.06763)  
- **FABind+**:   
	[![Code](https://img.shields.io/github/stars/QizhiPei/FABind?style=for-the-badge&logo=github)](https://github.com/QizhiPei/FABind)  
	[![Last Commit](https://img.shields.io/github/last-commit/QizhiPei/FABind?style=for-the-badge&logo=github)](https://github.com/QizhiPei/FABind)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:N/A-blue?style=for-the-badge&logo=arxiv)](https://arxiv.org/abs/2310.06763)  
- **GAABind**: GAABind is a Geometry-Aware Attention-Based Network for accurate protein-ligand binding pose and binding affinity prediction, featuring a comprehensive environment setup and dataset processing guide.  
	[![Code](https://img.shields.io/github/stars/Mercuryhs/GAABind?style=for-the-badge&logo=github)](https://github.com/Mercuryhs/GAABind/blob/main/README.MD)  
	[![Last Commit](https://img.shields.io/github/last-commit/Mercuryhs/GAABind?style=for-the-badge&logo=github)](https://github.com/Mercuryhs/GAABind/blob/main/README.MD)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:1-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bib/bbad462)  
- **GNINA**: GNINA is a molecular docking program that incorporates scoring and optimization of ligands using convolutional neural networks, aiming to combine the versatility of smina and AutoDock Vina with the predictive power of deep learning.  
	[![Code](https://img.shields.io/github/stars/gnina/gnina?style=for-the-badge&logo=github)](https://github.com/gnina/gnina)  
	[![Last Commit](https://img.shields.io/github/last-commit/gnina/gnina?style=for-the-badge&logo=github)](https://github.com/gnina/gnina)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:228-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1186/s13321-021-00522-2)  
- **GNINA_KD**:   
	[![Code](https://img.shields.io/github/stars/YanjingLiLi/GNINA_Knowledge_Distillation?style=for-the-badge&logo=github)](https://github.com/YanjingLiLi/GNINA_Knowledge_Distillation)  
	[![Last Commit](https://img.shields.io/github/last-commit/YanjingLiLi/GNINA_Knowledge_Distillation?style=for-the-badge&logo=github)](https://github.com/YanjingLiLi/GNINA_Knowledge_Distillation)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=arxiv)](https://doi.org/10.26434/chemrxiv-2024-0jh8g)  
- **gnina-torch**: A PyTorch implementation of the GNINA molecular docking scoring function, designed for enhanced performance and adaptability.  
	[![Code](https://img.shields.io/github/stars/RMeli/gnina-torch?style=for-the-badge&logo=github)](https://github.com/RMeli/gnina-torch/tree/0.0.2?tab=readme-ov-file)  
	[![Last Commit](https://img.shields.io/github/last-commit/RMeli/gnina-torch?style=for-the-badge&logo=github)](https://github.com/RMeli/gnina-torch/tree/0.0.2?tab=readme-ov-file)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.5281/zenodo.6943066)  
- **Interformer**: code to be released, a unified model built upon the Graph-Transformer architecture, which specially crafted to capture non-covalent interactions through the interaction-aware mixture density network  
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](http://dx.doi.org/10.21203/rs.3.rs-3995849/v1)  
	[![Link](https://img.shields.io/badge/Link-offline-red?style=for-the-badge&logo=xamarin&logoColor=red)](https://assets.researchsquare.com/files/rs-3995849/v1_covered_1a7bca9a-ea43-4846-aade-5187b4ca1ab6.pdf?c=1711449538)  
- **KarmaDock**:   
	[![Code](https://img.shields.io/github/stars/schrojunzhang/KarmaDock?style=for-the-badge&logo=github)](https://github.com/schrojunzhang/KarmaDock/blob/main/README.md)  
	[![Last Commit](https://img.shields.io/github/last-commit/schrojunzhang/KarmaDock?style=for-the-badge&logo=github)](https://github.com/schrojunzhang/KarmaDock/blob/main/README.md)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:26-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1038/s43588-023-00511-5)  
- **PLANTAIN**: Predicting LigANd pose wiTh an AI scoring functioN  
	[![Code](https://img.shields.io/github/stars/molecularmodelinglab/plantain?style=for-the-badge&logo=github)](https://github.com/molecularmodelinglab/plantain)  
	[![Last Commit](https://img.shields.io/github/last-commit/molecularmodelinglab/plantain?style=for-the-badge&logo=github)](https://github.com/molecularmodelinglab/plantain)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:N/A-blue?style=for-the-badge&logo=arxiv)](https://arxiv.org/abs/2307.12090)  
- **PointVS**: SE(3)-equivariant point cloud networks designed for virtual screening, enabling E(3)-invariant predictions of binding pose and affinity using networks based on the EGNN graph neural network layer.  
	[![Code](https://img.shields.io/github/stars/jscant/PointVS?style=for-the-badge&logo=github)](https://github.com/jscant/PointVS)  
	[![Last Commit](https://img.shields.io/github/last-commit/jscant/PointVS?style=for-the-badge&logo=github)](https://github.com/jscant/PointVS)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1101/2022.10.28.511712)  
- **SurfDock**: Surface-Informed Diffusion Generative Model for Reliable and Accurate Protein-ligand Complex Prediction  
	[![Code](https://img.shields.io/github/stars/CAODH/SurfDock?style=for-the-badge&logo=github)](https://github.com/CAODH/SurfDock)  
	[![Last Commit](https://img.shields.io/github/last-commit/CAODH/SurfDock?style=for-the-badge&logo=github)](https://github.com/CAODH/SurfDock)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:1-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1101/2023.12.13.571408)  
- **SurfDock**: A Surface-Informed Diffusion Generative Model for reliable and accurate protein-ligand complex prediction, integrating generative model techniques for enhanced docking predictions.  
	[![Code](https://img.shields.io/github/stars/CAODH/SurfDock?style=for-the-badge&logo=github)](https://github.com/CAODH/SurfDock)  
	[![Last Commit](https://img.shields.io/github/last-commit/CAODH/SurfDock?style=for-the-badge&logo=github)](https://github.com/CAODH/SurfDock)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:1-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1101/2023.12.13.571408)  
- **TankBind**: Trigonometry-Aware Neural NetworKs for Drug-Protein Binding Structure Prediction  
	[![Code](https://img.shields.io/github/stars/luwei0917/TankBind?style=for-the-badge&logo=github)](https://github.com/luwei0917/TankBind)  
	[![Last Commit](https://img.shields.io/github/last-commit/luwei0917/TankBind?style=for-the-badge&logo=github)](https://github.com/luwei0917/TankBind)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:49-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1101/2022.06.06.495043)  
- **TopoFormer**: A topological transformer for protein-ligand complex interaction prediction, integrating multiscale topology techniques with a structure-to-sequence transformer model.  
	[![Code](https://img.shields.io/github/stars/WeilabMSU/TopoFormer?style=for-the-badge&logo=github)](https://github.com/WeilabMSU/TopoFormer)  
	[![Last Commit](https://img.shields.io/github/last-commit/WeilabMSU/TopoFormer?style=for-the-badge&logo=github)](https://github.com/WeilabMSU/TopoFormer)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.5281/zenodo.10892800.svg)  
- **Uni-Dock**: GPU-Accelerated Docking Enables Ultralarge Virtual Screening (standalone).  
	[![Code](https://img.shields.io/github/stars/dptech-corp/Uni-Dock?style=for-the-badge&logo=github)](https://github.com/dptech-corp/Uni-Dock)  
	[![Last Commit](https://img.shields.io/github/last-commit/dptech-corp/Uni-Dock?style=for-the-badge&logo=github)](https://github.com/dptech-corp/Uni-Dock)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:22-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jctc.2c01145)  
- **Uni-Mol**: A Universal 3D Molecular Representation Learning Framework  
	[![Code](https://img.shields.io/github/stars/deepmodeling/Uni-Mol?style=for-the-badge&logo=github)](https://github.com/deepmodeling/Uni-Mol)  
	[![Last Commit](https://img.shields.io/github/last-commit/deepmodeling/Uni-Mol?style=for-the-badge&logo=github)](https://github.com/deepmodeling/Uni-Mol)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:N/A-blue?style=for-the-badge&logo=bookstack)](https://openreview.net/forum?id=6K2RM6wVqKu)  
- **Uni-Mol v2**:   
	[![Code](https://img.shields.io/github/stars/deepmodeling/Uni-Mol?style=for-the-badge&logo=github)](https://github.com/deepmodeling/Uni-Mol)  
	[![Last Commit](https://img.shields.io/github/last-commit/deepmodeling/Uni-Mol?style=for-the-badge&logo=github)](https://github.com/deepmodeling/Uni-Mol)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:N/A-blue?style=for-the-badge&logo=bookstack)](https://openreview.net/forum?id=6K2RM6wVqKu)  
	[![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://bohrium.dp.tech/apps/unimoldockingv2)  
- **vScreenML**: A machine learning classifier designed for virtual screening, allowing for the rescoring of hits to eliminate false positives, based on the Dataset of Congruent Inhibitors and Decoys (D-COID).  
	[![Code](https://img.shields.io/github/stars/karanicolaslab/vScreenML?style=for-the-badge&logo=github)](https://github.com/karanicolaslab/vScreenML)  
	[![Last Commit](https://img.shields.io/github/last-commit/karanicolaslab/vScreenML?style=for-the-badge&logo=github)](https://github.com/karanicolaslab/vScreenML)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1101/2020.01.10.902411v1.full.pdf)  
- **FeatureDock**: Protein-Ligand Docking Guided by Physicochemical Feature-Based Local Environment Learning using Transformer  
	[![Code](https://img.shields.io/github/stars/xuhuihuang/featuredock?style=for-the-badge&logo=github)](https://github.com/xuhuihuang/featuredock)  
	[![Last Commit](https://img.shields.io/github/last-commit/xuhuihuang/featuredock?style=for-the-badge&logo=github)](https://github.com/xuhuihuang/featuredock)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=arxiv)](https://doi.org/10.26434/chemrxiv-2024-dh2rw)  
- **GalaxyDock-DL**: Protein–Ligand Docking by Global Optimization and Neural Network Energy  
	[![Code](https://img.shields.io/github/stars/seoklab/galaxydock_dl?style=for-the-badge&logo=github)](https://github.com/seoklab/galaxydock_dl)  
	[![Last Commit](https://img.shields.io/github/last-commit/seoklab/galaxydock_dl?style=for-the-badge&logo=github)](https://github.com/seoklab/galaxydock_dl)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jctc.4c00385)  
- **LigPose**:   
	[![Code](https://img.shields.io/badge/Code)](https://gitfront.io/r/LigPose/kMWuV4DW6JpE/LigPose4Review/)
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.48550/arXiv.2408.11356)  

## **MetalloProteins**
- **MetalDock**:   
	[![Code](https://img.shields.io/github/stars/MatthijsHak/MetalDock?style=for-the-badge&logo=github)](https://github.com/MatthijsHak/MetalDock)  
	[![Last Commit](https://img.shields.io/github/last-commit/MatthijsHak/MetalDock?style=for-the-badge&logo=github)](https://github.com/MatthijsHak/MetalDock)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:4-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.3c01582)  

## **Multi-Ligand**
- **HARMONICFlow**:   
	[![Code](https://img.shields.io/github/stars/HannesStark/FlowSite?style=for-the-badge&logo=github)](https://github.com/HannesStark/FlowSite)  
	[![Last Commit](https://img.shields.io/github/last-commit/HannesStark/FlowSite?style=for-the-badge&logo=github)](https://github.com/HannesStark/FlowSite)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:N/A-blue?style=for-the-badge&logo=arxiv)](https://arxiv.org/abs/2310.05764)  

## **Negative Image based**
- **PANTHER**: Negative Image based docking and scoring  
	[![Publication](https://img.shields.io/badge/Publication-Citations:33-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1007/s10822-015-9870-3)  
	[![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://www.medchem.fi/panther/)  
- **O-LAP**:   
	[![Code](https://img.shields.io/github/stars/jvlehtonen/overlap-toolkit?style=for-the-badge&logo=github)](https://github.com/jvlehtonen/overlap-toolkit)  
	[![Last Commit](https://img.shields.io/github/last-commit/jvlehtonen/overlap-toolkit?style=for-the-badge&logo=github)](https://github.com/jvlehtonen/overlap-toolkit)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1186/s13321-024-00857-6)  

## **Pose Optimisation**
- **DeepRMSD-Vina**: DeepRMSD+Vina is a computational framework integrating ligand binding pose optimization and screening, utilizing deep learning alongside the classical Vina scoring function.  
	[![Code](https://img.shields.io/github/stars/zchwang/DeepRMSD-Vina_Optimization?style=for-the-badge&logo=github)](https://github.com/zchwang/DeepRMSD-Vina_Optimization)  
	[![Last Commit](https://img.shields.io/github/last-commit/zchwang/DeepRMSD-Vina_Optimization?style=for-the-badge&logo=github)](https://github.com/zchwang/DeepRMSD-Vina_Optimization)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:26-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bib/bbac520)  

## **Protein Docking**
- **EquiDock**: EquiDock employs geometric deep learning for fast and accurate rigid 3D protein-protein docking, focusing on efficiency and accessibility with comprehensive preprocessing and training guidelines.  
	[![Code](https://img.shields.io/github/stars/octavian-ganea/equidock_public?style=for-the-badge&logo=github)](https://github.com/octavian-ganea/equidock_public)  
	[![Last Commit](https://img.shields.io/github/last-commit/octavian-ganea/equidock_public?style=for-the-badge&logo=github)](https://github.com/octavian-ganea/equidock_public)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.48550/arXiv.2111.07786)  
- **LightDock**: The open-source macromolecular docking framework written in Python  
	[![Code](https://img.shields.io/github/stars/lightdock/lightdock?style=for-the-badge&logo=github)](https://github.com/lightdock/lightdock)  
	[![Last Commit](https://img.shields.io/github/last-commit/lightdock/lightdock?style=for-the-badge&logo=github)](https://github.com/lightdock/lightdock)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:82-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bioinformatics/btx555)  
- **PyDock3**: Electrostatics and desolvation for effective scoring of rigid-body protein-protein docking  
	[![Code](https://img.shields.io/github/stars/brianjimenez/pydock_tutorial?style=for-the-badge&logo=github)](https://github.com/brianjimenez/pydock_tutorial)  
	[![Last Commit](https://img.shields.io/github/last-commit/brianjimenez/pydock_tutorial?style=for-the-badge&logo=github)](https://github.com/brianjimenez/pydock_tutorial)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:282-blue?style=for-the-badge&logo=bookstack)](http://dx.doi.org/10.1002/prot.21419)  

## **Quantum**
- **Zhang et al.**:   
	[![Publication](https://img.shields.io/badge/Publication-Citations:1-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jctc.3c00943)  

## **Reviews**
- **Revolutionizing drug discovery: an AI-powered transformation of molecular docking**:   
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1007/s00044-024-03253-9)  

## **Shape-based**
- **PheSA**: An Open-Source Tool for Pharmacophore-Enhanced Shape Alignment  
	[![Code](https://img.shields.io/github/stars/joewah/PheSAExamples?style=for-the-badge&logo=github)](https://github.com/joewah/PheSAExamples)  
	[![Last Commit](https://img.shields.io/github/last-commit/joewah/PheSAExamples?style=for-the-badge&logo=github)](https://github.com/joewah/PheSAExamples)  
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1021/acs.jcim.4c00516)  

## **Template**
- **FitDock**: fits initial conformation to the given template using a hierachical multi-feature alignment approach, subsequently explores the possible conformations, and finally outputs refined docking poses  
	[![Publication](https://img.shields.io/badge/Publication-Citations:65-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/bib/bbac087)  
	[![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](http://cao.labshare.cn/fitdock/php/index.php)  

## **Water**
- **WatVina**: Watvina facilitates drug design with support for explicit or implicit waters, pharmacophore, or position-constrained docking, and external torsion parameters, enhancing the Autodock Vina engine.  
	[![Code](https://img.shields.io/github/stars/biocheming/watvina?style=for-the-badge&logo=github)](https://github.com/biocheming/watvina)  
	[![Last Commit](https://img.shields.io/github/last-commit/biocheming/watvina?style=for-the-badge&logo=github)](https://github.com/biocheming/watvina)  

## **WebServers**
- **MolModa**: accessible and secure molecular docking in a web browser.  
	[![Publication](https://img.shields.io/badge/Publication-Citations:1-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1093/nar/gkae406)  
	[![Link](https://img.shields.io/badge/Link-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://durrantlab.com/molmoda)  
- **TOMEv3**:   
	[![Publication](https://img.shields.io/badge/Publication-Citations:0-blue?style=for-the-badge&logo=bookstack)](https://doi.org/10.1016/j.jmb.2024.168704)  
	[![Webserver](https://img.shields.io/badge/Webserver-online-brightgreen?style=for-the-badge&logo=cachet&logoColor=65FF8F)](https://atome.cbs.cnrs.fr/ATOME_V3/index.html)  
